```{r introduction-setup, include=FALSE, echo = FALSE}
knitr::opts_chunk$set(echo = FALSE, dpi = 320, cache = TRUE,
                      message = FALSE,
                      fig.width = 8, fig.height = 8,
                      out.width = "80%", fig.align = 'center')
# packages
library(countrycode)
library(cowplot)
library(geodata)
library(ggsflabel)
library(here)
library(rnaturalearth)
library(sf)
library(tidyverse)
```


# Introduction {#introduction}


## Preface {-}


Sections of this introductory chapter have contributed to first authored and co-authored peer-reviewed articles. The section on Lassa fever case-fatality rate and biases in reporting I published as a single author paper in a modified format in *International Health* [@simons_lassa_2022]. The section on rodent hosts of *Lassa mammarenavirus* I contributed to a co-authored paper on One Health approaches to Lassa fever surveillance and control in *Annals of Clinical Microbiology and Antimicrobials* [@arruda_niche_2021]. Components of the section on Lassa fever epidemiology and the section on rodent hosts of LASV relating to phylogenetics has been incorporated into an article on the sampling and sequencing biases of *Lassa mammarenavirus* which has been pre-printed on *MedRxiv* and is currently under review [@arruda_current_2023].


## Zoonotic infectious diseases


Zoonotic infectious diseases - or "zoonoses" - in humans are caused by pathogens transmitted either directly (e.g., bites or scratches) or indirectly (e.g., via vectors, environmental or food contamination) from animal hosts, including livestock, wildlife, and pets [@world_health_organization_taking_2019]. Zoonoses include bacterial, fungal, parasitic and viral microorganisms. Within their animal hosts, zoonotic pathogens do not always cause clinical disease. For example, *Lassa mammarenavirus* (LASV), the causative agent of Lassa fever in humans is not considered to cause significant clinical disease in rodent host species as measured through organ dysfunction, weight loss or behavioural change [@safronetz_temporal_2022]. However, in humans LASV infection can lead to severe clinical symptoms and death [@thielebein_virus_2022]. In contrast, Highly Pathogenic Avian Influenza, caused by *Influenza A virus* (subtype H5N1), leads to significant morbidity and mortality in some infected bird species (i.e., *Galliformes sp.*), alongside pathogenicity in humans during rare zoonotic spillover events [@haider_unusually_2017; @writing_committee_of_the_second_world_health_organization_consultation_on_clinical_aspects_of_human_infection_with_avian_influenza_a_h5n1_virus_update_2008]. 


The wider term “zoonotic disease” is often used for a disease that first originated in non-human animals and may continue to be used, even when disease transmission is no longer dependent on an animal reservoir (e.g., HIV, SARS-CoV-2) [@kock_situation_2022]. Here, individual transmission events from vertebrate animal populations into human populations - "spillover events" - can lead to, secondary epidemiological cycles, epidemics and pandemics  [@plowright_pathways_2017]. This is different to human infection from a zoonosis where in each case the animal transmits the infection to the human causing disease [@kock_situation_2022]. The patterns of spillover can also differ across zoonoses. For example, Nipah virus infection (*Nipah henipavirus*) and LASV spillover events from wild animal sources occur at relatively frequent intervals but result in limited, onward human-to-human transmission leading to small-sized, geographically constrained outbreaks of human disease [@luby_recurrent_2009; @lo_iacono_using_2015]. However, Nipah virus infection differs importantly from LASV with most human outbreaks associated with direct infection from an amplifying intermediate host, rather than direct infection from the reservoir species. In contrast to both LASV and Nipah virus, Ebola virus disease (*Sudan ebolavirus* and *Zaire ebolavirus*) and Mpox (formerly Monkeypox caused by the *Mpox virus*) exhibit sustained human-to-human transmission following spillover, but due to the transmission dynamics of these pathogens, outbreaks are generally constrained to local epidemics [@legrand_understanding_2007; @fine_transmission_1988].

Some pathogens may be better adapted to transmission among humans due to their specific properties or similarities between human physiology or immunology and those of the primary vertebrate reservoir  (e.g., HIV and SARS-CoV-2). Such pathogens are able to rapidly expand beyond the geographic region of the initial spillover event via human transmission chains and may be described as zoonotic diseases with no further important transmission from wild or domestic animal populations [@ye_zoonotic_2020; @marx_aids_2004]. Spillover may not be limited to a single direction of animal to human transmission and "spillback" (i.e., reverse zoonosis or zooanthroponosis) can potentially play important roles in maintaining pathogen endemicity with subsequent "secondary spillover" into human populations [@kreuder_johnson_spillover_2015]. Further, spillback can lead to morbidity and mortality in animal populations [@fagre_assessing_2022].


The different patterns of spillover are observable through phylogenetic analysis of viral sequences from human populations. For example, phylogenetic analysis of the Middle East respiratory syndrome coronavirus (MERS-CoV) suggests initial spillover events into human populations from Dromedary camels, which may be the reservoir host or an intermediate host, with subsequent establishment in the local human population ultimately leading to international epidemics, most notably in 2013 [@corman_rooting_2014; @haagmans_middle_2014]. Similarly, the multi-country Mpox outbreak in 2022 has been maintained by sustained human-to-human transmission originating within the endemic region, either directly linked to a spillover event or cryptic (i.e., unobserved) transmission among local human populations [@isidro_phylogenomic_2022]. In contrast, phylogenetic analysis of LASV sequences indicate that the most common recent ancestor of viruses currently circulating in Nigeria originated >1000 years prior, while sequences from Guinea and Sierra Leone suggest a more recent introduction 220 and 150 years ago respectively [@andersen_clinical_2015]. The interpretation of studies on LASV phylogenetics are consistent with repeated spillover events into human populations from pathogens circulating within a single or multiple reservoir species [@andersen_clinical_2015; @kafetzopoulou_metagenomic_2019; @villabona-arenas_phylogenetic_2020]. While the 2022 Mpox outbreak and ongoing SARS-CoV-2 pandemic are important examples of zoonoses causing epidemics and pandemics beyond their host species' ranges, these remain relatively rare events when compared to recurrent spillover events within endemic regions [@dudas_mers-cov_2018; @lloyd-smith_epidemic_2009]. The example of LASV highlights the risk of endemic zoonoses leading to recurrent local spillover events into human populations in endemic regions and reinforces the importance of surveillance of known zoonoses.


When considering interventions to reduce the health impact of zoonoses in endemic settings (e.g., through reducing the risk of recurrent local spillover events), an approach that incorporates knowledge of multiple interacting systems are required. Understanding the role of environmental, wildlife and human factors on the risk of spillover events are necessary. This is often termed the "One Health" framework: a "collaborative, multisectoral, and transdisciplinary approach - working at the local, regional, national and global levels - with the goal of achieving optimal health outcomes recognizing the interconnection between people, animals, plants, and their shared environment." [@noauthor_one_2022]. This framework is particularly useful when considering changes in risk from zoonoses occurring in the context of  ongoing climate, land use and biodiversity change.


## Global change and zoonoses


Anthropogenic climate change has long been known to modify the risk of zoonoses to human populations through several mechanisms [@jones_zoonosis_2013; @daszak_anthropogenic_2001]. For example, changes in mean temperature and precipitation will alter environmental suitability for both pathogens and hosts leading to expansion or contraction of endemic regions [@mills_potential_2010]. In addition, environmentally transmitted zoonoses such as *Leptospira spp.* (causing Leptospirosis), become better able to persist in the environment under changes that increase ambient temperature in the presence of increased precipitation, leading to higher prevalence and incidence of infection [@lau_climate_2010; @llop_prediction_2022]. Vector borne zoonoses such as West Nile Virus are currently demonstrating range expansion as both mosquito vector abundance and occurrence is increased across a larger geographic range, likely due to a combination of warmer winter periods, increased precipitation and a higher prevalence of potential breeding sites [@farooq_artificial_2022; @hoover_west_2016]. 

Climate change is occurring in step with anthropogenic land use change. Human driven conversion of natural landscapes towards human dominated use occurs at both a local and global scale through direct and indirect human actions (i.e., agricultural development, natural resource extraction, and urbanisation) [@gottdenker_anthropogenic_2014]. The association of land use change and pathogen transmission is complex, with increasing, decreasing and no change in pathogen transmission reported from observational studies of pathogen systems [@gottdenker_anthropogenic_2014]. Encroachment of human activity into zoonotic host animal ranges, as can occur under conditions of land use change, has been hypothesised to increase the risk of spillover events into human populations, through increasing the animal-human interface and raising the probability of direct and indirect contact with infected hosts of zoonoses [@murray_human_2013]. Additionally, heightened interactions between wildlife and domesticated animals  as a consequence of land use change and progression to intensive livestock production may also increase the risk of subsequent zoonosis into human populations, where wild sylvatic animals are hosts of pathogens that can be amplified in domesticated animals (e.g., Nipah and Hendra virus) [@plowright_ecological_2015; @epstein_nipah_2006]. In tandem, climate and land use change can also modify species' home ranges [@sultaire_climate_2016; @brodie_synergistic_2016]. As a consequence, an increased frequency of contact events between current and potential future hosts of zoonoses are produced, increasing the potential for cross-species pathogen transmission and the subsequent expansion of a zoonosis' endemic range [@carlson_climate_2022]. This has been observed with regards to Hendra virus, where Southern range expansion of the black fruit bat (*Pteropus alecto*) has resulted in domesticated horses in Australia being infected, with subsequent spillover events into human populations [@yuen_hendra_2021].


Animal biodiversity (or lack thereof) has also been proposed to modulate zoonosis risk, with several mechanisms proposed. The "Dilution effect" - initially applied to the Lyme disease system (*Borrelia burgdorferi sensu lato*), which comprises several vectors and animal hosts - hypothesises that in settings of low species diversity (operationalised as species richness), infection rates increase in a host species. The inverse scenario is one in which higher levels of animal biodiversity is protective to human health through reducing the rate of zoonosis into human populations [@ostfeld_biodiversity_2000]. This theory has been supported by studies of several pathogen systems across parasites, bacteria, viruses and fungi [@keesing_impacts_2010; @civitello_biodiversity_2015]. There is ongoing debate as to whether the Dilution effect is a general property of zoonosis systems, with several studies suggesting the inverse. This opposing mechanism, termed the "Amplification effect", occurs when increasing biodiversity, particularly through introduction of a new or more competent host species can increase the rate of infection in hosts and potentially the risk of zoonosis [@johnson_parasite_2012; @halliday_multivariate_2017]. These two effects may exist as a spectrum where dominance of one over the other is dependent on the specific disease context, with limited generalisability between disease systems [@gomez-hernandez_concurrent_2023].


The relationship between biodiversity and zoonosis risk has been advanced by consideration of the changes in species composition of an animal community rather than a single summary measure of biodiversity. A synthesis of the effect of land use change on biodiversity across multiple spatial scales and zoonosis systems found that species richness of zoonotic pathogen host species, but not non-host species, increased along an anthropogenic land use gradient [@gibb_zoonotic_2020]. Using this model anthropogenic land use change is proposed moderate zoonosis risk through biodiversity by restructuring species communities towards those that are able to persist in modified environments. These species are, on average, more likely to be hosts of zoonoses [@han_rodent_2015; @albery_fast-lived_2021].


Climate, land use and biodiversity change are interacting components within an ecosystem and attributing an effect of each independently to the risk of zoonosis is challenging [@gibb_ecosystem_2020]. It should also be noted that the observed land use changes are occurring at different rates globally, which may complicate current findings. Climate, land use and biodiversity change occurring in regions associated with a greater diversity of known zoonotic pathogens may potentially have a greater impact on the risk of zoonosis than in settings of low diversity of zoonotic pathogens.


## Zoonoses and host discovery: sampling considerations


Most microorganisms are non-pathogenic to humans or animals and provide vital ecosystem services. The small subset of microorganisms (<1%) that are pathogenic are typically able to replicate in multiple hosts [@editors_microbiology_2011; @woolhouse_diseases_2001; @woolhouse_population_2001]. It has been estimated that 60% of emerging human infectious diseases are associated with zoonotic pathogens (defined as those with a non-human animal source)  [@jones_global_2008]. However, only a minority of these (>200 pathogens) currently fulfil the WHO definition of a zoonosis a "disease or infection that is naturally transmissible from vertebrate animals to humans" [@world_health_organisation_zoonoses_2023]. Zoonoses are known to exist in the majority of terrestrial mammal orders (21/27) with the number of hosts of zoonotic pathogens strongly positively associated with the species richness of these orders [@han_global_2016]. However, the discovery of zoonoses is variable across mammalian taxa, with sampling efforts increased in orders with increased human interaction or of special interest (i.e., primates and livestock species). A recently compiled dataset (CLOVER) contains an increased number of documented pathogens in Primates, Artiodactyla (ungulates) and Carnivora alongside Rodentia and Chiroptera [@gibb_data_2021; @gibb_viralemergenceclover_2021] (Figure \@ref(fig:host-pathogen-associations)) Of these, Rodentia contain the greatest number of pathogens known to be zoonotic [@han_global_2016]. 


```{r host-pathogen-associations, out.width = "100%", message = FALSE, warning = FALSE, fig.scap = '(ref:short-1)', fig.cap = '(ref:fig-1-long)'}

if(!file.exists(here("thesis", "introduction_files", "clover.rds"))) {
  
  bacteria <- read_csv(gzcon(url("https://github.com/viralemergence/clover/raw/main/clover/clover_1.0_allpathogens/CLOVER_1.0_Bacteria_AssociationsFlatFile.csv")), show_col_types = FALSE)
  other <-  read_csv(gzcon(url("https://github.com/viralemergence/clover/raw/main/clover/clover_1.0_allpathogens/CLOVER_1.0_HelminthProtozoaFungi_AssociationsFlatFile.csv")), show_col_types = FALSE)
  viruses <- read_csv(gzcon(url("https://github.com/viralemergence/clover/raw/main/clover/clover_1.0_allpathogens/CLOVER_1.0_Viruses_AssociationsFlatFile.csv")), show_col_types = FALSE)
                 
  clover <- list(bacteria = bacteria,
                 other = other,
                 viruses = viruses) %>%
    bind_rows()
  
  all_h_p <- clover %>%
    filter(HostClass == "mammalia") %>%
    filter(PathogenNCBIResolved == TRUE) %>%
    select(HostOrder, HostGenus, Host, PathogenType, Pathogen, ReleaseYear, PublicationYear) %>%
    mutate(year = pmin(ReleaseYear, PublicationYear, na.rm = TRUE)) %>%
    select(HostOrder, HostGenus, Host, PathogenType, Pathogen, year) %>%
    group_by(HostGenus, Host, PathogenType, Pathogen, year) %>%
    distinct() %>%
    group_by(HostOrder, HostGenus, Host, PathogenType, Pathogen) %>%
    arrange(year) %>%
    slice(1)
  
  summary_all <- all_h_p %>%
    distinct(HostOrder, Host, PathogenType, Pathogen) %>%
    group_by(HostOrder, Host, PathogenType, Pathogen) %>%
    summarise(N = n()) %>%
    group_by(HostOrder, PathogenType) %>%
    mutate(N_host = n_distinct(Host)) %>%
    group_by(HostOrder, PathogenType, N_host) %>%
    summarise(N = sum(N)) %>%
    arrange(-N)
  
  rodentia_clover <- all_h_p %>%
    filter(HostOrder == "rodentia")
  
  clover_data <- list(summary_all = summary_all,
                      rodentia_clover = rodentia_clover)
  
  write_rds(clover_data, here("thesis", "introduction_files", "clover.rds")) 
  
} else {
  
  clover_data <- read_rds(here("thesis", "introduction_files", "clover.rds"))
}

known_pathogens <- clover_data$summary_all %>%
  ungroup() %>%
  mutate(HostOrder = fct_inorder(str_to_sentence(HostOrder)),
         PathogenType = fct_inorder(str_to_sentence(str_replace_all(PathogenType, "/", " and ")))) %>%
  ggplot() +
  geom_col(aes(x = HostOrder, fill = PathogenType, y = N)) +
  geom_label(aes(x = HostOrder, y = N_host, label = N_host), size = 3, label.padding = unit(0.05, "lines"),
             label.r = unit(0, "lines"), label.size = NA) +
  scale_y_continuous(trans = "log10") +
  scale_fill_brewer(palette = "Dark2") +
  coord_flip() +
  facet_wrap(~ PathogenType) +
  theme_bw() +
  labs(x = "Host order",
       fill = "Pathogen group",
       y = "Number of pathogens",
       title = element_blank()) +
  theme(legend.position = "none")


known_pathogens
```

(ref:short-1) Number of pathogens associated with each order of Mammalia.

(ref:fig-1-long) Number of pathogens identified from each Mammalian order. Bars indicate the number of known pathogens within different mammalian orders contained within CLOVER; the values within the bars indicate the number of distinct species within the order known to host these pathogens. Data obtained from CLOVER (Gibb et al. 2021).

Two mammalian taxa, Rodentia and Chiroptera are associated with the greatest number of species  that are hosts of zoonoses and overall number of zoonoses [@han_rodent_2015]. It is unclear whether these taxa represent special reservoirs that lead to an increased proportion of zoonotic viruses circulating within these species or make them more likely to transmit pathogens to humans, or whether the increased number of zoonoses associated with these taxa is driven by their increased species richness [@olival_are_2012; @wolfe_origins_2007; @luis_comparison_2013; @mollentze_viral_2020].

Discovery of zoonoses is biased both by the ability to detect them with current diagnostic assays and the sampling effort within different animal species and geographic regions [@grange_ranking_2021; @gibb_mammal_2021]. The discovery rate of viral zoonoses, an important subset of all zoonoses, has increased with improvements in the technical means to detect and identify them [@woolhouse_temporal_2008]. For example, the number of discovered rodent pathogens doubled since 2001 (\@ref(fig:rodent-pathogen-discovery)). The rate of discovery has exceeded previous expectations of global viral biodiversity, but continues to remain taxonomically and geographically biased, thus limiting the inferences that can be made for the risk of spillover events drawing on current data sources [@wille_how_2021]. Similar limitations are likely for other zoonoses taxa including bacteria, fungi and parasites.

```{r rodent-pathogen-discovery, out.width = "100%", message = FALSE, warning = FALSE, fig.scap = '(ref:short-2)', fig.cap = "Number of identified pathogens in the order Rodentia. Discovery of pathogens in rodents, the order containing the greatest number of zoonotic pathogens, has increased over time. 1,199 rodent pathogens are contained within CLOVER up to 2016. The number of pathogens discovered in each year were extracted from CLOVER and subset by pathogen family. Data obtained from CLOVER (Gibb et al. 2021)."}

rodent_pathogen_discovery <- clover_data$rodentia_clover %>%
  ungroup() %>%
  mutate(PathogenType = fct(str_to_sentence(str_replace_all(PathogenType, "/", " and ")), levels = levels(known_pathogens$data$PathogenType))) %>%
  ggplot() +
  geom_bar(aes(x = year, fill = PathogenType)) +
  scale_fill_brewer(palette = "Dark2") +
  facet_wrap(~ PathogenType, ncol = 1) +
  theme_bw() +
  guides(fill = "none") +
  labs(x = "Year",
       y = "Number of pathogens",
       title = element_blank())

rodent_pathogen_discovery
```

(ref:short-2) The number of known pathogens associated with the mammalian order Rodentia.

## Rodent associated zoonoses


A growing body of evidence is highlighting the importance of rodents as a key reservoir for known and expected zoonoses. Rodents are a diverse, globally distributed mammalian order that provide important and beneficial ecosystem services including pest regulation and seed dispersal [@fischer_ecosystem_2018]. Of the almost 2,600 species, representing 40% of all mammalian species, 282 species (~11%) have been identified to be reservoirs of 95 known zoonoses, a greater number than other mammal orders [@delia_rodent_2019; @ecke_population_2022; @han_global_2016]. The majority of these zoonoses are viruses (34) and bacteria (26) with the remaining including helminths, protozoa and fungi, representing a minority of all rodent pathogens (\@ref(fig:rodent-pathogen-discovery)). As discussed above the high prevalence in this order may be driven by high species richness, rather than any inherent properties of the order Rodentia [@mollentze_viral_2020]. 


Within this order, the prevalence of zoonoses are disproportionally high within species that demonstrate "fast" life history strategies, although the effect of sampling biases and confounding effects such as synanthropy (an organism that lives near and benefits from humans and their environmental modifications) may be producing some of this observed effect [@han_rodent_2015; @albery_fast-lived_2021]. Fast-lived rodent species (i.e., those prioritising reproduction over survival and longevity), are typically small, abundant and are more commonly urban-adapted [@albery_fast-lived_2021]. These species favour inexpensive, nonspecific immune defenses (e.g., anatomic barriers, nonspecific inhibitors and phagocytosis), which make them more likely to be hosts of pathogens, including zoonoses, although whether these properties are consistent within genera is unclear and whether these findings are replicated in wild, as opposed to laboratory, animals is unknown [@dianzani_nonspecific_1996; @martin_immune_2007; @viney_immunology_2017]. Independently of the causal drivers of high zoonoses prevalence among rodent species, components of their life histories, such as high population density, increase the risk of spillover into human populations. 


Synanthropy is common among rodent species, more so among rodent species known to be reservoirs of zoonoses [@ecke_population_2022]. Synanthropic species tend to be highly abundant in locations in which they occur, with high population densities and dynamic population fluctuations in response to resource availability, which promotes both frequency- and density-dependent transmission of pathogens among hosts [@ecke_population_2022]. The high abundance of these species in human dominated landscapes increases the rate of contact with humans providing increased opportunities for both direct- and indirect transmission of rodent associated zoonoses [@morand_changing_2019; @iacono_unified_2016].


Rodent species that have wide ranges may display heterogeneity across their range in both their biology and behaviour. For example, studies in *Clethrionomys* voles, hosts of Puumala orthohantavirus, have been observed to display different population dynamics across a latitudinal gradient from Northern Finland to Central Europe, affecting pathogen dynamics within these populations [@turchin_empirically_1997; @henttonen_small_2001]. Similarly, while the primary reservoir species of LASV, *Mastomys natalensis*, has been observed to have dramatic population fluctuations in the Eastern extent of its range (i.e., Tanzania), the same amplitude of population fluctuations have not been observed in West African populations (i.e., Guinea) where they host LASV [@fichet-calvet_reproductive_2008; @leirs_stochastic_1997].  This may impact the generalisability of studies conducted in specific locations within a rodent's range when attempting to understand the broader geographic risk of rodent associated zoonosis.


## West Africa is a geographic hotspot for zoonosis risk


Geographic hotspots of zoonotic disease risk are predicted to occur where mammalian host species richness is greatest, such as in the tropics [@han_global_2016]. West Africa is one such location of high mammalian biodiversity [@ceballos_global_2006]. This region is also undergoing significant anthropogenic change, driven by increasing human populations, agricultural development, urbanisation and resource extraction (e.g., deforestation and mining) alongside effects of anthropogenic climate change such as desertification and changes in precipitation dynamics [@haggblade_regulating_2022; @maconachie_diamond_2012; @walther_urbanisation_2021; @nicholson_desertification_1998; @bongaarts_human_2009]. It has also been the location of several recent zoonosis epidemics and outbreaks, for example, the 2014 Ebola epidemic and ongoing Lassa fever outbreaks. 


While the number of zoonotic infectious disease outbreaks and, associated human morbidity and mortality is rising in West Africa, it is imperative to consider these trends in the local context of anthropogenic change described above, particularly as the number of people at risk of infection is continuing to increase [@makoni_africas_2020]. Alongside these global changes, improved pathogen discovery, increased access to diagnostics and healthcare, and a higher level of case reporting may result in an apparent increase in the burden of zoonotic infectious diseases in the region. An example of intensifying pathogen discovery is the Pandemic Preparedness For Global Health Security (PREDICT) programme, conducted between 2009 and 2020, which tested in excess of 164,000 samples from animals and humans in 14 African countries and 12 Asian countries identified 949 novel viruses including 217 known zoonoses [@noauthor_about_2019; @amman_isolation_2020]. Projects such as PREDICT can importantly change our understanding of the prevalence and locations of zoonoses, although these pathogens have likely circulated in the region for many years prior to discovery. Improved diagnostics and reporting of zoonoses are evident in the case of Lassa fever, particularly in Nigeria. Here, the Nigeria Center for Disease Control (NCDC) has prioritised the expansion of diagnostic testing capability. Prior to 2005, molecular diagnosis of Lassa fever infection was not possible in Nigeria with samples transferred to the Lassa fever unit at Kenema General Hospital, Sierra Leone [@naidoo_nigerias_2020]. Between 2005 and 2012 testing was established in Lagos and Irrua, Nigeria with further laboratory capacity established at the National Reference Laboratory in Abuja and in Ebonyi state in 2018. The expansion of testing capacity has led to in excess of 20,000 individuals being tested for Lassa fever between 2018 and 2021. As such, trends in the number of reported cases of Lassa fever from Nigeria may not be reflective of a true increase in cases [@dalhat_epidemiological_2022].


The detection of zoonotic infectious disease outbreaks typically relies upon clinical case detection of infected humans within healthcare settings (i.e., real-time surveillance and reporting) rather than monitoring transmission among wild or domestic animals (i.e., zoonotic disease surveillance). No public health system has to date implemented active surveillance systems through testing of animal populations in West Africa. Elsewhere (e.g., in Europe), active surveillance in birds and horses is conducted for West Nile Virus to inform risk assessments of human disease outbreaks [@wendt_zoonotic_2015; @gossner_west_2017]. The Global Health Security Index measured activities conducted by countries to assess their ability to respond to a potential emerging outbreak of a zoonotic infectious disease [@global_health_security_index_ghs_2022]. Across West Africa real-time surveillance and reporting of zoonosis was rated as poor, with the notable exception of Nigeria, suggesting that these countries may not be able to rapidly identify outbreaks of endemic zoonotic diseases of epidemic potential (i.e., Ebola, Mpox and Lassa fever) (Figure \@ref(fig:global-zoonotic-disease-surveillance)A). No countries in the region were found to have integrated zoonotic disease surveillance among animal host species with public health institutions (Figure \@ref(fig:global-zoonotic-disease-surveillance)B). Zoonosis surveillance in animals in West Africa is currently limited to academic or programmatic research which has been used by public health agencies to aid risk stratification of patients that present with symptoms consistent with these diseases, based on when, where, and why they present to local healthcare services [@leski_sequence_2015; @happi_increased_2022].

```{r global-zoonotic-disease-surveillance, out.width = "100%", warning = FALSE, fig.scap = '(ref:short-3)',fig.cap = "Global Health Security Index country scores for the sub-domains of A) Real-time surveillance and reporting and B) Zoonotic disease surveillance. Data obtained from the Global Health Security Index with the score for the sub-domains shown for the countries of West Africa. Labels refer to the three letter ISO codes for countries: Benin (BEN), Burkina Faso (BFA), Côte d'Ivoire (CIV), Ghana (GHA), Guinea (GIN), Gambia (GMB), Guinea-Bissau (GNB), Liberia (LBR), Mali (MLI), Mauritania (MRT), Niger (NER), Nigeria (NGA), Senegal (SEN), Sierra Leone (SLE) and Togo (TGO)."}

ghs_data <- read_csv(here("thesis", "introduction_files", "ghs_index.csv"), show_col_types = FALSE)

ghs_data <- ghs_data %>%
  mutate(iso3c = countrycode(Country, origin = "country.name", destination = "iso3c")) %>%
  left_join(codelist %>%
              select(iso3c, un.region.name, region23), by = "iso3c") %>%
  mutate(Value = cut(Value, breaks = c(0, 25, 50, 75, 100), include.lowest = TRUE))

world <- ne_countries(scale = 110, returnclass = "sf") %>%
  select(adm0_a3, admin)

fig_1_3a <- ghs_data %>%
  left_join(world, by = c("iso3c" = "adm0_a3")) %>%
  filter(region23 == "Western Africa" &
           Factor == "Real-time Surveillance and Reporting") %>%
  st_as_sf() %>%
  ggplot() +
  geom_sf(aes(fill = Value)) +
  geom_sf_text(aes(label = iso3c)) +
  scale_fill_viridis_d(option = "cividis", direction = -1,
                       limits = levels(ghs_data$Value)) +
  labs(fill = "GHS Index score",
       title = "A) Real-time surveillance and reporting",
       x = element_blank(),
       y = element_blank()) +
  theme_bw()

fig_1_3b <- ghs_data %>%
  left_join(world, by = c("iso3c" = "adm0_a3")) %>%
  filter(region23 == "Western Africa" &
           Factor == "Zoonotic Disease Surveillance - Animals") %>%
  st_as_sf() %>%
  ggplot() +
  geom_sf(aes(fill = Value)) +
  geom_sf_text(aes(label = iso3c)) +
  scale_fill_viridis_d(option = "cividis", direction = -1,
                       limits = levels(ghs_data$Value)) +
  labs(fill = "GHS Index score",
       title = "B) Zoonotic disease surveillance - animals",
       x = element_blank(),
       y = element_blank()) +
  theme_bw()

leg <- get_legend(fig_1_3a +
                    theme(legend.position = "bottom"))

plot_grid(plotlist = list(fig_1_3a +
                            theme(legend.position = "none"),
                          fig_1_3b +
                            theme(legend.position = "none"),
                          leg),
          ncol = 1,
          rel_heights = c(1, 1, 0.2))
```

(ref:short-3) Global Health Security scores for sub-domains associated with zoonosis surveillance.

## Lassa fever: A case study of a rodent associated zoonosis in West Africa


In West Africa the morbidity and mortality caused by the rodent associated zoonosis Lassa fever is of particular importance. Despite the discovery of LASV more than 50 years ago and the expected hundreds of thousands of annual human cases research into this disease system adopting a One Health framework is currently limited. Specifically, human communities in Sierra Leone have been inordinately impacted by recurrent outbreaks of this disease. Lassa fever also serves as a useful case study of rodent associated zoonoses given the interaction between multiple small mammal species', potential roles of climate change and land use change on disease risk and the relatively high frequency of spillover events.


### *Lassa mammarenavirus* and Lassa fever


*Lassa mammarenavirus* (LASV) is an enveloped, bisegmented, single stranded RNA virus of the Arenaviridae family. It is a zoonotic pathogen and an established zoonosis, causing Lassa fever in humans. Lassa fever is a potentially lethal viral haemorrhagic fever, first identified from a case series of infected patients seeking healthcare in Jos, Nigeria in 1969,  [@frame_lassa_1970]. Human LASV infection is caused by spillover of the virus from infected rodents and their excreta, with a limited role of human-to-human secondary transmission [@mccormick_prospective_1987; @lo_iacono_using_2015]. Human-to-human transmission is typically reported from nosocomial settings with limited evidence for transmission within households; no transmission chains beyond two cases have been reported (i.e., an index case to another susceptible human) [@garry_lassa_2023]. The reservoir host of LASV has been identified as the multimammate rat (*M. natalensis*) following an outbreak in Sierra Leone between 1970-2 [@monath_lassa_1974]. This synanthropic rodent species is found across much of sub-Saharan Africa. However, outside West Africa, no individuals of this species have been found to be infected with LASV [@grobbelaar_mammarenaviruses_2021; @bellocq_dhati_2020; @colangelo_mitochondrial_2013].


*Lassa mammarenavirus* has four confirmed lineages (I-IV) and three additional lineages (V-VII) based on geographic and phylogenetic analysis [@li_genetic_2023]. Lineages I, II, III and VI are located within Nigeria; lineage IV contains all isolates from the Mano River region of Guinea, Liberia and Sierra Leone; lineage V contains samples from Mali and the Ivory Coast; and lineage VII contains recently sampled sequences from Togo [@ehichioya_phylogeography_2019; @andersen_clinical_2015; @manning_lassa_2015; @whitmer_new_2018]. Lineage I is believed to be the most ancient, originating around 1,000 years ago in the North East of Nigeria, with subsequent radiation and establishment of lineages II and III in the Southern and Central areas of the country, respectively [@ehichioya_phylogeography_2019; @andersen_clinical_2015]. Lineage IV represents a westward expansion of the virus into the Mano River region, dated around 350 years ago [@andersen_clinical_2015].


Host cell entry of the virus is mediated by a trimeric glycoprotein complex that interacts with host cell receptors and leads to fusion of the viral and host membranes, *in vivo* this protein undergoes substantial host-derived glycosylation, effectively reducing available antibody binding domains [@hastie_lassa_2018]. Once within the host cell, the viral nucleoprotein associates with viral RNAs forming ribonucleoprotein complexes facilitating transcription and replication of viral RNA within the host cell cytoplasm [@hass_replicon_2004]. The process of viral entry into host cells is expected to lead to the observed tissue tropism (i.e., dendritic cells) in experimental infection models in guinea pigs and *M. natalensis* [@torriani_lassa_2017]. Within infected guinea pigs and *M. natalensis*, LASV load was highest transiently in the lymph nodes with sustained high titres in the lungs and spleen [@jahrling_pathogenesis_1982; @safronetz_temporal_2022]. Minimal pathological changes were observed in the lungs or spleen of infected guinea pigs or *M. natalensis*, with no evidence of clinical disease in these animals.


Among infected humans with clinical symptoms, the viral incubation period is between 7 and 18 days [@mccormick_prospective_1987]. The route of infection is unknown, although inhalation of aerosolised viral particles and direct contact with infectious material are suspected [@monath_lassa_1975; @stephenson_effect_1984]. Initial symptoms are non-specific with fever, weakness, malaise, cough, sore throat and a typically frontal headache [@knobloch_clinical_1980]. The majority of symptomatic patients will go on to develop joint and lumbar pain, a non-productive cough with many developing severe retrosternal chest pain, nausea with vomiting and diarrhoea and abdominal pain [@mccormick_lassa_2002]. Up to a third of hospitalised patients will significantly decline 6-8 days post onset of fever with a minority developing a haemorrhagic syndrome with bleeding from the mucosal surfaces. Severe pulmonary oedema and soft tissue oedematous changes in the head and neck are common in fatal cases [@knobloch_clinical_1980]. The vast majority of infections, commonly reported as 80%, are asymptomatic although in the absence of long term prospective studies, the proportion of asymptomatic infections is difficult to estimate [@mccormick_prospective_1987]. There is some limited evidence, based on geographic differences in case fatality rates (CFRs), that disease severity may vary by infecting lineage with the highest CFRs reported from Sierra Leone (Lineage IV) [@garry_lassa_2023]. There is no evidence on the association of disease severity or pathogenicity in humans with the route of infection.


Several risk factors for severe disease in humans have been identified. For example, infection is considered particularly lethal in pregnant women during the third trimester of pregnancy with infection usually fatal for the developing foetus [@okogbenin_retrospective_2019]. Age stratified risk is perhaps more complex with worse outcomes for those over 45 years old in Nigeria, whereas more fatal cases are reported in children, teenagers or young adults in Sierra Leone [@okokhere_clinical_2018; @shaffer_space-time_2021]. These discrepancies may be related to different circulating LASV lineages (i.e., Lineage I, II, III and VI in Nigeria and Lineage IV in Sierra Leone), or differences in case detection and healthcare seeking behaviour between the countries.


Treatment options for acute cases of Lassa fever are limited. Ribavirin is the standard of care for treating acute cases although the effectiveness of this treatment is questionable [@salam_ribavirin_2022]. Supportive care therefore remains the mainstay of treatment for hospitalised individuals. There are no currently available vaccinations for Lassa fever, although three candidate vaccines have begun Phase I and II clinical trials [@salami_review_2019; @inovio_pharmaceuticals_dose-ranging_2022; @inovio_pharmaceuticals_study_2020; @international_aids_vaccine_initiative_phase_2023; @themis_bioscience_gmbh_randomized_2022].


The CFR of Lassa fever has been reported to be as high as 29.7% although this varies by country and year [@kenmoe_systematic_2020]. This estimate is based on a systematic review of the published scientific literature and does not include data from epidemiological reports or WHO bulletins. My recent review derived an estimated CFR of 16.5% (+/- 5%) among confirmed cases [@simons_lassa_2022]. Importantly, this estimate is sensitive to biases in reporting and is likely a grossly inflated rate of mortality. Severe cases are more likely to come into contact with healthcare services and be tested for Lassa fever, and these cases are also more likely to result in disease-associated mortality skewing confirmed cases to those with severe disease. Therefore, this CFR should be considered a severe disease CFR, with the majority of mildly symptomatic cases likely to have a dramatically reduced probability of mortality, which will lower the total CFR and increase the number of underreported cases. Currently, a lack of disease surveillance and monitoring data prevents the estimation of the infection-fatality rate for Lassa fever.


Survivors of symptomatic Lassa fever may have lasting effects of the disease. Sensorineural hearing loss is reported to occur in up to a third of Lassa fever survivors and potentially causes significant social and public health burden in the region that have not been well studied [@mateer_lassa_2018]. Additional neurological sequealae reported in Lassa fever survivors include cerebellar ataxia and visual impairment, although few patients have been assessed for these complications and progression over time is unclear [@li_ophthalmic_2020; @ezeomah_sequelae_2019]. Most hospitalised patients, following recovery, rapidly clear viral RNA from sera (typically within 10 days post-hospitalisation) [@thielebein_virus_2022]. Most patient sera are negative for viral RNA at hospital discharge, however, up to 50% of male survivors have detectable viral loads in seminal fluid at 3 months post-hospitalisation raising concerns that human-to-human sexual transmission may be possible [@thielebein_virus_2022]. LASV has also been detected in the breast milk of Lassa fever survivors suggesting potential vertical transmission to offspring from infected nursing mothers [@salu_monitoring_2020].


### Lassa fever epidemiology


Annual Lassa fever incidence is unknown, with estimates ranging between 150,000 to 900,000 cases per year annually [@mccormick_prospective_1987; @basinski_bridging_2021]. The wide uncertainty surrounding these estimates is due to a combination of few serological studies, limited disease surveillance and an overlap between the symptomatology of Lassa fever with other infectious diseases in these endemic regions (e.g., malaria, Rift Valley fever). Lassa fever is currently considered endemic in 8 West African countries: Benin, Ghana, Guinea, Liberia, Mali, Nigeria, Sierra Leone and Togo by the World Health Organisation (WHO), with sporadic cases reported from Burkina Faso and the Ivory Coast (Figure \@ref(fig:lassa-endemic)) [@world_health_organisation_lassa_2022]. Countries designated as endemic by the WHO do not all regularly reported cases of Lassa fever (e.g., Guinea) and between endemic countries there are substantial differences in the number of annually reported cases [@simons_lassa_2022]. The endemic region is entirely contained within the range of the reservoir species *M. natalensis*. Imported cases have been reported from non-West African countries such as the United Kingdom, Germany and the United States of America with a limited number of secondary human-to-human transmission events observed outside the endemic region [@tuite_potential_2019; @wolf_fifty_2020].


```{r lassa-endemic, warning = FALSE, message = FALSE, echo = FALSE, out.width = "100%", fig.scap = '(ref:short-4)', fig.cap = '(ref:long-1)'}

continental_wa <- c("BEN", "BFA", "CIV", "GHA", "GIN", "GMB", "GNB", "LBR", "MLI", "NER", "NGA", "SEN", "SLE", "TGO")
endemic_lassa <- c("BEN", "GHA", "GIN", "LBR", "MLI", "NGA", "SLE", "TGO")
outbreak_lassa <- c("BFA", "CIV")

# Download shapefiles from GADM and save them
wa_shape <- gadm(country = continental_wa, level = 0, path = here("thesis", "introduction_files", "wa_gadm"), version = "latest") %>%
  st_as_sf() %>%
  select(GID_0, COUNTRY)

wa_region <- wa_shape %>%
  filter(GID_0 %in% continental_wa) %>%
  mutate(status = factor(case_when(GID_0 %in% endemic_lassa ~ "Endemic",
                            GID_0 %in% outbreak_lassa ~ "Outbreaks",
                            TRUE ~ "No cases"), levels = c("Endemic", "Outbreaks", "No cases")))

wa_region$COUNTRY[wa_region$COUNTRY == "Côte d'Ivoire"] <- "Ivory Coast"

mastomys_range <- read_rds(here("thesis", "introduction_files", "mastomys_iucn.rds")) %>%
  st_crop(st_bbox(wa_region))

ggplot() +
  geom_sf(data = wa_region, aes(fill = status)) +
  geom_sf(data = mastomys_range, linetype = 2, lwd = 2, fill  = NA, colour = "black") +
  geom_sf_label_repel(data = wa_region, aes(label = COUNTRY)) +
  scale_fill_viridis_d(option = "cividis") +
  theme_bw() +
  theme(legend.position = "bottom") +
  labs(x = element_blank(),
       y = element_blank(),
       fill = "Lassa endemicity")


```

(ref:short-4) The Lassa fever endemic region.

(ref:long-1) The endemic region of Lassa fever and range of *Mastomys natalensis* in West Africa. The black dashed border indicates the range of *Mastomys natalensis* in West Africa, its range extends East and South across the continent (not shown here). Adapted from WHO Data on Lassa fever endemicity and the range of *Mastomys natalensis* from the International Union for Conservation of Nature Red List.

Nigeria and Sierra Leone have historically reported the greatest number of Lassa fever cases (Figure \@ref(fig:lassa-cases)). This is likely driven by increased availability of testing for acute cases in these countries. Human seroepidemiological surveys in Guinea, Mali and the Ivory Coast - countries that have generally reported few acute cases - report seroprevalence in excess of 20%, which suggests undetected localised transmission of LASV [@kerneis_prevalence_2009; @bausch_lassa_2001; @sogoba_lassa_2016; @safronetz_annual_2017; @akoua-koffi_detection_2006]. The number of reported cases across the region declined during the Ebola and SARS-CoV-2 epidemic where both healthcare seeking behaviour and the availability of Lassa fever testing may have reduced. The number of cases reported in Nigeria has generally increased since data became routinely available in 2012. In contrast, there has been a dramatic fall in cases reported from Sierra Leone since 2012. Whether these trends represent actual changes in the underlying spillover risk remains unclear.


```{r lassa-cases, warning = FALSE, message = FALSE, echo = FALSE, out.width = "100%", fig.scap = '(ref:short-5)', fig.cap = '(ref:long-epi)'}

lassa_raw <- read_csv(here("thesis", "introduction_files", "confirmed_cases.csv"))

lassa_clean <- lassa_raw %>%
  group_by(year, country) %>%
  mutate(n_records = n()) %>%
  ungroup() %>%
  filter(!(country == "nigeria" & region == "all" & n_records > 1)) %>%
  mutate(country = str_to_title(str_replace_all(country, "cote d'ivoire", "ivory coast")),
         GID_0 = countrycode(sourcevar = country, origin = "country.name", destination = "iso3c"),
         status = factor(case_when(GID_0 %in% endemic_lassa ~ "Endemic",
                                   GID_0 %in% outbreak_lassa ~ "Outbreaks",
                                   TRUE ~ "No cases"), levels = c("Endemic", "Outbreaks", "No cases")),
         country_label = factor(case_when(country %in% c("Benin", "Ghana", "Guinea", "Liberia", "Mali",
                                                         "Nigeria", "Sierra Leone", "Togo") ~ country,
                                          TRUE ~ "Combined"),
                                levels = c("Benin", "Ghana", "Guinea", "Liberia", "Mali",
                                           "Nigeria", "Sierra Leone", "Togo", "Combined"))) %>%
  group_by(year, country, country_label, status) %>%
  summarise(n_cases = sum(confirmed_cases, na.rm = TRUE))

colours_wa <- c("Endemic countries" = "white",
                "Benin" = "#871a10",
                "Ghana" = "#000000",
                "Guinea" = "#be2a2c",
                "Liberia" = "#0a2864",
                "Mali" = "#50ab4a",
                "Nigeria" = "#398053",
                "Sierra Leone" = "#2f71c1",
                "Togo" = "#f7d147",
                " " = "white",
                "Non-endemic countries" = "white",
                "Ivory Coast" = "#e6853b",
                "Burkina Faso" = "#107d87")

lassa_clean$country = factor(lassa_clean$country, levels=c("Endemic countries", "Benin", "Ghana", "Guinea",
                                                           "Liberia", "Mali", "Nigeria", "Sierra Leone", "Togo", " ",
                                                           "Non-endemic countries", "Ivory Coast", "Burkina Faso"))

ggplot(lassa_clean, aes(x = year, y = n_cases, fill = country)) +
  geom_col() +
  scale_x_continuous(breaks = seq(from = min(lassa_clean$year), to = max(lassa_clean$year), by = 4)) +
  scale_y_continuous(breaks = function(x) unique(floor(pretty(seq(0, (max(x) + 1) * 1.1))))) +
  scale_fill_manual(values = colours_wa,
                    drop = FALSE) +
  guides(fill=guide_legend(ncol = 1)) +
  theme(legend.key = element_rect(fill = NA),
        legend.title = element_blank()) +
  facet_wrap(~ status + country_label, scales = "free_y") +
  annotate(geom = "rect", xmin = 2013, xmax = 2016, ymin = -Inf, ymax = Inf, alpha = 0.1) +
  annotate(geom = "rect", xmin = 2020, xmax = year(Sys.Date()), ymin = -Inf, ymax = Inf, alpha = 0.1) +
  annotate(geom = "rect", xmin = year(Sys.Date()), xmax = year(Sys.Date()) + 1, ymin = -Inf, ymax = Inf, alpha = 0.2, fill = "orange") +
  theme_minimal() +
  labs(x = element_blank(),
       y = "Confirmed cases",
       fill = element_blank())
```

(ref:short-5) Confirmed Lassa fever cases in West Africa 2008-2023.

(ref:long-epi) Confirmed Lassa fever cases from countries in West Africa 2008-2023. Confirmed cases show variability by year with the greatest number of cases reported from Nigeria, Sierra Leone and Liberia. Grey shaded regions represent periods of regional or global epidemics which may have affected Lassa fever reporting (i.e., the Ebola epidemic and SARS-CoV-2 pandemic). The yellow shaded region represents 2023 where an incomplete year is shown. Data current as of 2023-07-13. Data compiled from scientific publications, epidemiological reports, ProMED mail and WHO disease outbreak notifications and available online [@simons_lassa_2022-1].

The number of reported confirmed cases of Lassa fever in endemic countries is likely to be significantly underreported. Cases tend to occur in rural and remote locations where healthcare access is generally low, and financial and societal costs of accessing healthcare are relatively high while testing facilities are concentrated in large urban settings [@bhadelia_understanding_2019; @nnaji_deuce-ace_2021]. Additionally while clinicians in endemic settings have good awareness of symptoms that may indicate acute Lassa fever infection, access to testing and timely reporting were identified as factors that could lead to diagnostic delay, poor patient outcomes and delayed public health responses to outbreaks [@olowookere_diagnostic_2014; @rohan_beyond_2022]. An estimate of the degree of underreporting was conducted using reported Lassa fever disease associated mortality, assuming a consistent 16.5% case-fatality rate across the region. Using this approach Nigeria was found to report the highest proportion of all expected cases (63%) while countries with generally fewer observed outbreaks reported significantly fewer than expected cases (e.g., Ghana - 17%, Guinea - 25%) [@simons_lassa_2022].

Human seroepidemiological studies conducted in several regions of Sierra Leone suggest that despite the observed fall in human cases of disease, infection remains prevalent. For example, a study in Sierra Leone suggests that widespread transmission of LASV is occurring outside the traditionally considered endemic region of Eastern Sierra Leone [@grant_seroprevalence_2023]. A large-scale serological study conducted by the Coalition for Epidemic Preparedness Innovations across Benin, Guinea, Liberia, Nigeria and Sierra Leone to understand the prevalence of antibodies against LASV is now ongoing [@penfold_prospective_2023].


### Rodent hosts of *Lassa mammarenavirus*


While *M. natalensis* is considered the only reservoir of LASV 11 other rodent species have been found to be acutely infected or have antibodies to the virus [@monath_lassa_1974; @olayemi_new_2016; @demby_lassa_2001; @fichet-calvet_lassa_2014]. The role of the wider rodent species community in viral transmission in endemic areas is not currently well understood. Further, serological evidence exists for prior exposure to LASV in non-rodent species, including domestic dogs, non-human primates and shrews. The role of these species in the ecology and transmission of LASV is even less clear [@kenmoe_systematic_2020].


*Mastomys natalensis* is a synanthropic rodent species, native to Africa. This species is considered a pest species across much of its range, as it lives within and around human communities consuming grain within the fields and in stores [@swanepoel_systematic_2017]. The species demonstrates archetypal fast life history traits with rapid sexual maturity (4 months), short life span (<1 year) and large litter sizes (mean of 9 live offspring) [@coetzee_biology_1975; @albery_fast-lived_2021; @safronetz_establishment_2021]. The proportion of reproductively active individuals is observed to increase in the late wet season and early dry season with a nadir in the late dry season, leading to a population boom in the late wet season [@mlyashimbi_relationships_2018; @mayamba_population_2021]. Importantly, the majority of population dynamic studies in this species have been conducted in Tanzania, where abundance has been observed to be closely linked to food availability. However, the drivers of these population dynamics may not be as extreme in West Africa where the population dynamics are less closely linked to rainfall patterns [@olayemi_small_2018; @fichet-calvet_reproductive_2008; @bangura_lassa_2021]. 


As a synanthropic species *M. natalensis* typically occurs within areas of human habitation and agriculture and is found to be an early invader of land converted to agricultural use [@makundi_reproduction_2007]. This land use preference is consistent across the entirety of its range with few individuals trapped in forested landscapes [@olayemi_small_2018; @fichet-calvet_reproductive_2008; @bangura_lassa_2021; @coetzee_biology_1975; @leirs_spatial_1996]. This would suggest that abundance of this species is heterogeneous across its proposed range with expected absence in the forested regions of sub-Saharan Africa. This species is non-territorial, co-existing with conspecifics and other rodent species and with a limited home range of ~30m, although dispersal across greater distances has been observed [@leirs_spatial_1996]. Contact with other rodent species is therefore assumed to be common and is supported by the high rodent species richness in locations where *M. natalensis* is detected [@fichet-calvet_reproductive_2008; @bangura_lassa_2021]. The high frequency of contacts may be potentially important for the transmission of LASV among rodent and non-rodent hosts as it potentially facilitates transmission of the virus across the heterogeneous land use types of the endemic region.


While this species is distributed across sub-Saharan Africa genomic studies suggest that six phylogroups (A-I to A-III and B-IV to B-VI) have formed which correspond to different geographic regions of Africa. The West African clade, A-I is genetically distinct and is found in the endemic region of LASV [@colangelo_mitochondrial_2013]. Individuals of the other clades have not been found to be infected with LASV but have tested positive for other Arenaviridae, including Mayo Ranewo (A-II), Dhati Welel (A-III), Gairo (B-IV), Morogoro (B-V), and Mopeia viruses (B-VI) [@bellocq_dhati_2020]. The presence of clade specific Arenaviridae may provide cross-immunity between these viruses, preventing of range expansion of LASV [@rodas_mucosal_2004]. Cross-immunity and the impact of this on the range of LASV has not been investigated in real-world settings, further studies are therefore required to understand the spatial interactions of different Arenaviridae. Further, differential susceptibility of the different phylogroups to individual Arenaviridae may explain why the Lassa fever endemic region is constrained to the Western radiation of *M. natalensis* despite it being found throughout sub-Saharan Africa and may limit any future geographic expansion of the virus. Importantly, there is currently no evidence to demonstrate that different phylogroups of *M. natalensis* display differential susceptibility to Arenaviridae.


While LASV is considered to primarily infect *M. natalensis* the prevalence of LASV in rodent and small-mammal communities varies importantly across the region and over time (Figure \@ref(fig:rodent-lassa-seroprevalence)). When detected LASV prevalence varies both within and between countries. In Guinea, LASV was detected in 10 to 55% of trapped individual rodents, from species including *Gerbilliscus kempi*, *Lemniscomys striatus*, *Lophuromys sikapusi*, *Mastomys erythroleucus*, *M. natalensis*, *Mus musculus*, *Mus minutoides*, *Praomys daltoni*, *Praomys rostratus* and *Rattus rattus* [@demby_lassa_2001; @fair_lassa_2007; @fichet-calvet_lassa_2014; @olayemi_new_2016; @marien_households_2020]. In a single study conducted in Mali, acute infection, identified through PCR, was detected in 25% of individuals of *M. natalensis*, while in, Sierra Leone, sampled rodents showed a wide range of prevalence from 1% to 100% among *M. natalensis* [@monath_lassa_1974; @mccormick_prospective_1987; @safronetz_detection_2010; @leski_sequence_2015]. However, most of the sampling events in these settings did not detect any acute infection (absence of acute infection not shown in Figure \@ref(fig:rodent-lassa-seroprevalence)). The number of individuals tested for acute infection is typically lower than those tested for antibodies due to availability of reagents, cost and laboratory requirements. There are also selection biases when rodents and small mammals are tested for acute infection which may increase the proportion of positive samples. For example, testing may only be performed in certain species, antibody positive small mammals or small mammals trapped in the location of a confirmed human case. 


```{r rodent-lassa-seroprevalence, warning = FALSE, message = FALSE, echo = FALSE, out.width = "100%", fig.scap = '(ref:short-6)', fig.cap = '(ref:long-prev)'}

review_data <- read_tsv(here("thesis", "introduction_files", "taxa.txt")) %>%
  filter(str_detect(specificEpithet, "Lassa") & occurrenceStatus == "Present")

lassa_testing <- review_data %>%
  mutate(rodent_species = str_remove_all(associatedTaxa, "parasite of "),
         n_tested = as.numeric(str_extract(occurrenceRemarks, "\\d+")),
         n_positive = as.numeric(organismQuantity),
         method = case_when(str_detect(identificationRemarks, "PCR|Culture") ~ "Acute infection",
                            str_detect(identificationRemarks, "Serology") ~ "Serology"),
         rodent_species = fct_rev(factor(str_to_sentence(case_when(rodent_species == "mastomys" ~ "mastomys spp.",
                                                           rodent_species == "crocidura" ~ "crocidura spp.",
                                                           rodent_species == "praomys" ~ "praomys spp.",
                                                           rodent_species == "rattus" ~ "rattus spp.",
                                                           rodent_species == "rodentia" ~ "rodentia spp.",
                                                           TRUE ~ rodent_species)),
                                 levels = c("Mastomys natalensis", "Mastomys erythroleucus", "Mastomys spp.", "Rattus rattus", "Rattus spp.", "Praomys daltoni", "Praomys rostratus", "Praomys spp.", "Gerbilliscus kempi", "Mus minutoides", "Mus baoulei", "Mus musculus", "Hylomyscus pamfi", "Lemniscomys striatus", "Crocidura spp.", "Lophuromys sikapusi", "Rodentia spp."))),
         perc_positive = round(n_positive/n_tested * 100, 1)) %>%
  select(country, rodent_species, perc_positive, n_tested, n_positive, method, datasetName, bibliographicCitation)

lassa_testing$perc_positive[lassa_testing$perc_positive == 200] <- 100

ggplot() +
  geom_point(data = lassa_testing %>%
               drop_na(rodent_species), aes(y = rodent_species, x = perc_positive, size = n_tested, colour = country), position = position_jitter(height = 0.2), alpha = 0.5) +
  scale_size(range = c(1, 6)) +
  scale_colour_brewer(palette = "Set1") +
  facet_grid(~ method) +
  theme_bw() +
  theme(axis.text.y = element_text(face = "italic")) +
  labs(y = "Genus/Species",
       x = "Percentage positive",
       colour = "Country",
       size = "Number assayed")

```


(ref:short-6) Prevalence of acute infection with LASV or antibodies to LASV among rodents sampled in West Africa.

(ref:long-prev) Prevalence of acute infection with LASV or antibodies to LASV in rodent and small-mammal species sampled in West Africa. The size of a point relates to the number of samples of that species tested and the colour to the country in which the rodent was sampled. Where possible the rodent species is identified, for individuals only identified to genus level the genera from which samples obtained is shown. Data obtained from: Monath *et al.*, 1974; Wulff *et al.*, 1975; McCormick *et al.*, 1987; Mahy, 1994; Demby *et al.*, 2001; Fair *et al.*, 2007; Safronetz *et al.*, 2010; Fichet-Calvet *et al.*, 2014; Leski *et al.*, 2015; Olayemi *et al.*, 2016; Karan *et al.*, 2019; Yadouleton *et al.*, 2019 and Mariën *et al.*, 2020. The percentage positive were calculated for each species or genus reported within the associated publication based on the number of individuals reported to be positive for PCR or serological assays out of all individuals tested.

*Mastomys erythroleucus*, one of the rodent species found to acutely infected with LASV is a morphologically indistinguishable, closely related species to *M. natalensis*. It has been found to co-occur with *M. natalensis* in several regions of Guinea, Sierra Leone and Nigeria [@brouat_phylogeography_2009]. The proportion of *M. erythroleucus* individuals found to be infected with LASV was not too dissimilar from *M. natalensis* and may indicate that this species can also be involved in viral transmission in locations where these species co-exist. Three other native rodent species *Mus minutoides*, *Mus baoulei* and *Hylomyscus pamfi* have been found to be acutely infected with LASV although the number of individuals of these tested was small. *Mus minutoides* and *Mus baoulei* are African pygmy mice - i.e., a complex of 17-19 morphologically similar rodents that may contain a number of undescribed subspecies [@britton-davidian_systematics_2012]. They occupy a wide range of land use types and are not considered synanthropic, with a habitat preference for forest and shrubland habitats, although they are often detected in cultivated landscapes [@long_multi-scale_2013]. Finally, the non-native, invasive rodent species, *Rattus rattus* has been found to be acutely infected with LASV. This synanthropic rodent species has been found to co-occur with *M. natalensis* in locations which it has invaded and may represent a relatively recent host of LASV within the endemic region [@olayemi_small_2018; @bangura_lassa_2021].

More rodent species have been found to have antibodies to LASV than those found to have acute infection. Whether these species are competent hosts of the virus and are able to produce subsequent rodent-to-rodent or rodent-to-human transmission is not known. Additionally, some of these detections may be due to the presence of cross-reactive antibodies to other Arenaviridae; however, a validated, commercial, ELISA assay used for many of these surveys shows a sensitivity of 97.1% and specificity of 100% to LASV [@soubrier_detection_2022]. The proportion of all samples that tested positive for LASV antibodies across all species is typically lower, consistent with the greater number of samples assayed and less targeted sampling. Additional species found to have antibodies to LASV include the native rodents *Praomys daltoni*, *Praomys rostratus*, *Lemniscomys striatus*, *Lophuromys sikapusi*, *Mus musculus* and *Gerbilliscus kempii*. Of these species, only *P. daltoni* and the invasive *M. musculus*, are considered synanthropic, typically found in villages and nearby agricultural areas in West Africa [@nicolas_comparative_2008; @diagne_serological_2017; @lippens_genetic_2017; @mikula_commensalism_2020]. *P. rostratus* and *L. sikapusi* are more commonly found in forested or fragmented forest, shrubland and agricultural habitats [@iyawe_distribution_1988; @akpatou_terrestrial_2018]. *L. striatus* and *G. kempii* are considered savannah rodents, rarely detected within villages but often detected in forested habitats, shrubland and agriculture [@davis_affinities_1949; @lourie_isolation_1975; @hoffmann_spatial_2001]. Finally, non-rodent, small mammal species found to have antibodies to LASV include individuals of the species rich insectivorous shrew order (*Crocidura*). As morphological identification to species level is typically not performed, the grouping is at the order level.


Two invasive rodent species, *Mus musculus* and *R. rattus* are increasingly common in West Africa. These species have been introduced through human activity, typically in coastal regions, beginning in the 15th century with subsequent expansion into the interior of countries along human transport networks [@dalecky_range_2015; @lippens_genetic_2017]. Populations of these species have been found to establish communities in areas of human habitation demonstrating their synanthropic properties [@hima_native_2019; @puckett_commensal_2020]. These species appear to have potentially different effects on local rodent species richness following establishment, with *M. musculus* but not *R. rattus* leading to reduced rodent species richness in locations in which it is detected [@dalecky_range_2015]. This may have important implications for the prevalence of LASV in the endemic region: if there is displacement of the primary reservoir by these invasive species that are potentially less competent hosts of viral transmission then the risk of Lassa fever outbreaks may subsequently decrease. As yet there is no direct evidence for the displacement of *M. natalensis* within the Lassa fever endemic region by invasive rodents or the effect this may have on LASV prevalence.


Sampling of rodent species and locations of confirmed human cases have been used to produce risk maps of Lassa fever outbreaks. Risk maps may be based on human cases, *M. natalensis* occurrence or a combination of both. These risk maps consistently identify the Mano River region and Nigeria as hotspots of risk [@fichet-calvet_risk_2009; @mylne_mapping_2015; @redding_environmental-mechanistic_2016; @basinski_bridging_2021]. The studies are conducted at the regional scale and are not able to incorporate the heterogeneity of rodent species occurrence or abundance that has been observed in rodent sampling studies. Additionally, the potential contribution of wider rodent communities to viral transmission or maintenance is not incorporated in these models as they all consider *M. natalensis* as the sole reservoir of LASV.


Importantly, sampling of rodents and small mammals for LASV in West Africa is biased taxonomically and geographically with increased sampling effort in locations reporting historical outbreaks of Lassa fever [@beck_spatial_2014; @klitting_predicting_2022]. Limiting sampling to these locations may be artificially biasing risk towards these regions. For example, a study accounting for some of these spatial sampling biases suggesting that risk is more evenly distributed across West Africa than what has historically been reported [@peterson_mapping_2014]. A better understanding of sampling biases in both small mammal communities and pathogen sampling will assist in identifying regions in which additional sampling is required. Further, predicting future change in risk that could ensue from ongoing global change will be similarly limited by data biases [@boria_spatial_2014; @wille_how_2021].


## Thesis aims and structure


To better understand the current and future risk of Lassa fever zoonosis in West Africa, biases in available data need to be characterised and systematic sampling of the entire rodent community and co-located small mammals is required. This thesis aimed to address some of the critical gaps in understanding of LASV transmission among rodent communities in endemic settings, whether there is a need to move away from the current single reservoir species paradigm and how to quantify the effect of anthropogenic change on the structure of rodent communities and how this may modulate Lassa fever spillover risk.


The first part of my thesis (Chapter \@ref(chapter2)) examines a key problem of sampling bias of both rodents and their pathogens across West Africa and whether these biases can be quantified. I synthesised rodent trapping studies in West Africa to assess spatial biases, identifying regions that have been relatively under-sampled and therefore locations in which inference may be limited based on available datasets. I used this dataset to then compare with a commonly used resource, the Global Biodiversity Information Facility (GBIF), to explore the benefit of incorporating primary rodent trapping data within this larger, consolidated dataset. The results I present in this chapter and the planned incorporation of this data into GBIF will aid researchers attempting to model risks of rodent associated zoonoses in West Africa and the effect of future anthropogenic change on rodent distributions across the region.


The second part of my thesis (Chapters \@ref(methodschapter), \@ref(chapter3) and \@ref(chapter4)) presents my systematic study of rodent ecology and LASV prevalence in an endemic region of Eastern Sierra Leone. Primary data collection was in the form of a two-and-half-year longitudinal rodent trapping study, comprising in excess of 43,000 trap nights (methods described in Chapter \@ref(methodschapter)). In chapter \@ref(chapter3), I describe the composition of small-mammal communities in the Eastern Province of Sierra Leone to examine whether small-mammal species richness and diversity vary along an anthropogenic land use gradient. I use these data to explore the biotic interactions between species to infer the risk of LASV transmission among small-mammal communities along a land use gradient using species occupancy models which account for incomplete detection. In chapter \@ref(chapter4), I expand on this work to explicitly model potential contact networks among individual species in different land use types to investigate the interactions within these rodent communities. The key research question I address is whether the primary reservoir is more likely to interact with members of the same species and what consequences this may have for viral transmission. I use Exponential-family Random Graph Models fitted to produced networks of rodent contacts based on rodent trapping data to assess the probability of inter- and intra-specific contact rates to understand viral transmission within rodent communities. I also describe the prevalence of antibodies to LASV to gauge the risk of Lassa fever spillover in these settings. 


My thesis concludes with a discussion of how insights from my work might enhance our understanding of the risk of rodent associated zoonoses in West Africa in general, and Lassa fever emergence in Sierra Leone in particular. I also discuss the future directions required to better quantify this dynamic risk, alongside how dynamic risk estimates can guide timely public health interventions to reduce disease associated morbidity and mortality. 


\newpage
