```{r introduction-setup, include=FALSE, echo = FALSE}
knitr::opts_chunk$set(echo = FALSE, dpi = 320, cache = TRUE,
                      fig.width = 8, fig.height = 8,
                      out.width = "80%", fig.align = 'center')
##packages
library(tidyverse)
library(here)
library(countrycode)
library(rnaturalearth)
library(sf)
```


# Introduction {#introduction}


## Zoonotic infectious diseases


Zoonotic infectious diseases are diseases of humans caused by pathogens transmitted from animal hosts. Specifically, a "zoonoses" is any disease or infection that is shared between animals - including livestock, wildlife, and pets - and people, either through direct or indirect pathways [@world_health_organization_taking_2019]. Examples of direct transmission include, bites and scratches (i.e., *Rabies lyssavirus* and *Bartonella hanselae*) while indirect transmission can occur via arthropod vectors (i.e., *Orentia tsustsugamushi*, Scrub typhus), environmental contamination (i.e., Leptospirosis) and through food (i.e., Salmonellosis). The wider term "zoonotic disease" is often used for a disease that first originated in non-human animals, even when disease transmission is currently entirely within the human population in the absence of animals or an animal reservoir (e.g., HIV) [@kock_2022]. Individual transmission events from vertebrate animal populations into human populations - spillover events - can, under certain circumstances, lead to sustained outbreaks that may progress to localised epidemics or global pandemics [@plowright_2017]. These zoonotic pathogens may - or may not - cause clinical disease in their animal hosts. For example, *Lassa mammarenavirus* (LASV), the causative agent of Lassa fever in humans is not thought to cause significant clinical disease in rodent host species' as measured through organ dysfunction, weight loss or behavioural change [@safronetz_temporal_2022]. Meanwhile, in humans LASV infection can lead to severe clinical symptoms and death [@safronetz_temporal_2022]. In contrast, Highly Pathogenic Avian Influenza, caused by *Influenza A virus* (subtype H5N1), leads to significant morbidity and mortality in infected bird species alongside pathogenicity in humans [@haider_unusually_2017; @writing_committee_of_the_second_world_health_organization_consultation_on_clinical_aspects_of_human_infection_with_avian_influenza_a_h5n1_virus_update_2008]. 


Zoonoses display a range of patterns of "spillover" from wild or domestic animals and transmission in human populations. Nipah virus infection (*Nipah henipavirus*) and Lassa fever (LASV) spillover into human populations from wild animal sources occur at relatively frequent intervals but result in limited, onward human-to-human transmission leading to small-sized, geographically constrained outbreaks of human disease [@luby_recurrent_2009; @lo_iacono_using_2015]. Ebola virus disease (*Sudan ebolavirus* and *Zaire ebolavirus*) and mpox (formerly Monkeypox caused by the *Mpox virus*) exhibit sustained human-to-human transmission following spillover from wild or domestic animals, but due to the transmission dynamics of these pathogens, outbreaks are generally constrained to local epidemics [@legrand_understanding_2007; @fine_transmission_1988]. Finally, some pathogens may be better adapted, or able to become better adapted, to transmission among humans due to their pathogen properties or similarities between human physiology and their primary vertebrate reservoir, these are able to rapidly expand beyond the geographic region of the initial spillover event via human transmission chains and may become zoonotic diseases with no further important transmission from wild or domestic animal populations (i.e., HIV and SARS-CoV-2) [@ye_zoonotic_2020; @marx_aids_2004]. There is also evidence to suggest that spillover may not be be limited to a single direction of animal to human and that "spillback" can play potentially important roles in maintaining pathogen endemicity with subsequent "secondary spillover" into human populations or can lead to morbidity and mortality in animal populations [@fagre_assessing_2022].


These different patterns of spillover can be observed through phylogenetic analysis of viral sequences from human populations. For example, phylogenetic analysis of the SARS-CoV-2 virus suggest an initial spillover event into human populations in October and November of 2019 with establishment in the local human population ultimately leading to a global pandemic beginning in 2020 [@pekar_timing_2021]. Similarly the multi-country mpox outbreak in 2022 is proposed to be secondary to human-to-human sustained transmission following a single origin endemic country, either directly linked to a spillover event or cryptic transmission among local human populations [@isidro_phylogenomic_2022]. A contrast to these recent outbreaks is observed in Lassa fever where phylogenetic analysis of LASV sequences indicated the most common recent ancestor of viruses circulating in Nigeria was >1000 years prior, while sequences from Guinea and Sierra Leone suggest a more recent introduction of 220 and 150 years respectively [@andersen_clinical_2015]. This findings are consistent with repeated spillover events into human populations from pathogens circulating within reservoir species. While the 2022 mpox outbreak and ongoing SARS-CoV-2 pandemic are important examples of zoonoses causing epidemics and pandemics beyond their host species' ranges, these remain relatively rare events when compared to recurrent spillover events within endemic regions [@dudas_mers-cov_2018; @lloyd-smith_epidemic_2009]. The situation of LASV highlights the ongoing hazard of local spillover into human populations in endemic regions and reinforces the importance of surveillance of known zoonoses.


When considering interventions to reduce the health impact of zoonoses in endemic settings an approach that incorporates knowledge of multiple interacting systems are required. Understanding the role of environmental, wildlife and human factors on the hazard and risk of spillover events are necessary. This is often termed the "One Health" approach, a "collaborative, multisectoral, and transdisciplinary approach - working at the local, regional, national and global levels - with the goal of achieving optimal health outcomes recognizing the interconnection between people, animals, plants, and their shared environment." [@CDC_2023]. This framework is particularly useful when considering how spillover of zoonoses occur in a setting of ongoing climate, landuse and biodiversity change.


## Global change and zoonoses


Anthropogenic climate change is hypothesised to modify the risk of zoonoses to human populations through several mechanisms [@jones_zoonosis_2013; @daszak_anthropogenic_2001]. Changes in mean temperature and precipitation will alter environmental suitability for both pathogens and hosts leading to expansion or contraction of endemic regions [@mills_potential_2010]. Environmentally transmitted zoonoses such as Leptospira will become better able to persist in the environment under changes that increase ambient temperature in the presence of increased precipitation, leading to increased prevalence and incidence of infection [@lau_climate_2010; @llop_prediction_2022]. Vector borne zoonoses such as West Nile Virus are currently demonstrating range expansion as both mosquito vector abundance and occurrence is increased across a larger geographic range [@farooq_artificial_2022; @hoover_west_2016]. 

Climate change is one component of anthropogenic global change but is occurring in step with anthropogenic landuse change. Human driven conversion of natural landscapes towards human dominated use occurs at both a local and global scale through direct and indirect human actions (i.e., agricultural development, natural resource extraction, and urbanisation) [@gottdenker_anthropogenic_2014]. The association of landuse change and pathogen transmission is complex, with increasing, decreasing and no change in pathogen transmission reported from primarily observational studies [@gottdenker_anthropogenic_2014]. Encroachment of human activity into zoonotic host animal ranges, as can occur under landuse change, has been hypothesised to increase the risk of spillover events into human populations, through increasing the animal-human interface raising the probability of direct and indirect contact with infected hosts of zoonoses [@murray_2013]. Additionally, increased interactions between wildlife and domesticated animals may also increase the risk of subsequent zoonosis spillover into human populations where wild sylvatic animals are hosts of pathogens that may be amplified in domesticated animals following transmission (i.e. Nipah and Hendra virus) [@plowright_ecological_2015; @epstein_nipah_2006]. Together climate and landuse change can also modify species' home ranges. This can drive contact events between current hosts of zoonoses and potential future hosts of a pathogen, increasing the potential for cross-species pathogen transmission and the subsequent expansion of a zoonoses' endemic range [@carlson_2022]. This phenomenon has been observed in Hendra virus where a Southern range expansion of the black fruit bat host (*Pteropus alecto*) has resulted in domesticated horses in Australia being infected, with subsequent spillover events into human populations [@yuen_2021].


An additional mechanism through which zoonosis spillover risk is modulated is animal biodiversity. Several mechanisms for the association of animal biodiversity and zoonosis risk have been proposed. The "Dilution effect" - initially applied to the Lyme disease (*Borrelia burgdorferi*) zoonosis system which comprises several vectors and animal hosts - posits that in settings of low species diversity (measured as species richness) that infection rates increase in a host species, the inverse being that higher levels of animal biodiversity reduces the rate of zoonosis spillover into human populations [@ostfeld_2000]. This theory has been supported by investigations of a range of zoonoses including parasites, bacteria, viruses and fungi [@keesing_2010; @civitello_2015]. There is ongoing debate as to whether this is a general property of zoonosis systems, as several studies have suggested the inverse, an "Amplification effect", where increasing biodiversity, particularly through introduction of a new host, or more competent host species can increase the rate of infection in hosts and potentially the risk of zoonosis spillover [@johnson_2012; @halliday_2017]. These two effects may exist as a spectrum where dominance of one over the other is dependent on the specific disease context [@gomez-hernandez_concurrent_2023].


Climate, landuse and biodiversity change are interacting components within an ecosystem and attributing an effect of each independently to the risk of zoonosis spillover is challenging [@gibb_2020]. A synthesis of the effect of landuse change on biodiversity across multiple scales and zoonosis systems found that species richness of host species of zoonosis increased but not non-host species, along an anthropogenic landuse gradient [@gibb_2020]. These changes are also occurring at different rates globally. Climate, landuse and biodiversity change occurring in regions associated with a greater diversity of known zoonotic pathogens may potentially have a greater impact on the risk of zoonosis spillover than in settings of low diversity of zoonotic pathogens.


## Locations of zoonoses


The majority of microorganisms are non-pathogenic to humans or animals and provide vital ecosystem services. The small subset of microorganisms (~1%) that are pathogenic are typically able to replicate in multiple hosts [@editors_microbiology_2011; @woolhouse_diseases_2001; @woolhouse_population_2001]. For example, of human emerging infectious diseases 60% are associated with zoonoses, therefore able to infect both animals and humans [@jones_global_2008]. Zoonoses are globally distributed, found on all continents. Zoonoses comprise, bacteria, fungi, parasites and viruses. The global virome is estimated at X species with Y expected to be zoonoses. The distribution of known zoonoses is biased by global sampling effort. Expected diversity of zoonoses is greatest in the tropics i.e., South America, Africa and South East Asia.


West Africa is a nexus of for global change in addition to increasing human population. The impact of known and novel zoonoses in this region requires further understanding. 
    + Rate of discovery   
    + Surveillance bias   
    + Reporting bias    
    
Viruses are important zoonoses.    
    + Mutation and diversity    
    + Treatment options   
    + Transmission potential    
    + Health burden   

## Rodent borne zoonoses


Zoonoses are observed in most animal orders. The orders with the highest number of known zoonoses are Chiroptera and Rodentia. Zoonoses from rodents are expected in West Africa.


Properties of these species support them being hosts of zoonoses.
    + Trait associations    
    + Immunology    
    + Host ranges   
    + Biodiversity and dilution   
    + Human-host interactions   

Synanthropy among hosts of zoonoses leads to increased hazard of spillover.
    + Commensalism    
    + Abundance   
    + Known zoonoses    


## Lassa fever

### *Lassa mammarenavirus* and Lassa fever

    + Describe virology and ecology   
    + Strains   
    + Epidemiology    
    + Ecology   
    + Viral lifecycle   
    + Risk factors of human infection 
    + Lassa fever pathology
    + Sequaelae of human infection   

### Lassa fever in Sierra Leone

    + Current risk    
    + Prior research
    
### Rodent hosts of *Lassa mammarenavirus*

    + Known reservoirs
    + Potential reservoirs
    
### Heterogeneity of rodent occurrence and abundance

    + Effect of landuse on occurrence
    

### Surveillance of endemic zoonotic infectious diseases

Currently detection of outbreaks of zoonotic infectious diseases relies primarily upon clinical case detection of infected humans rather than evidence of circulating transmission among wild or domestic animals. Few health systems utilise active surveillance systems with testing of animal populations, a notable exception is *West Nile virus* surveillance in birds and horses in Europe [@gossner_west_2017]. Here, surveillance in an animal-human-vector approach informs public health agencies to increased risk of human infection from this vector borne disease. While in West Africa, detection of outbreaks of endemic zoonotic infectious diseases such as, Ebola, Lassa fever, Monkeypox and Leptospirosis occurs following identification of human cases (Figure \@ref(fig:global-zoonotic-disease-surveillance)). Surveillance among host species is limited to academic or programmatic research which has been used to identify regions at potentially greater risk for spillover events, this information is then used to aid risk stratification of patients that present with symptoms consistent with these diseases, based on when in the year they present, the location from which they present and suspected risk behaviours [@leski_sequence_2015; @happi_increased_2022]. Fewer countries, with none in West Africa have surveillance systems that combine animal and human data [@wendt_zoonotic_2015].

Human cases presenting to healthcare are classified as suspected, possible, probable or confirmed cases of the disease of interest. This classification occurs based on clinical symptoms, disease progression and known risk factors. Individuals presenting to healthcare may become suspected cases in the context of a known outbreak or failure of treatment for more common infections, such as malaria and bacterial infections [reference needed]. Once suspicion is raised for a potential zoonotic infectious disease as a cause of presentation, individuals may be tested for known pathogens according to local guidance, the availability of this testing varies by location. In Nigeria, the Nigerian Centre for Disease Control have rapidly expanded testing capacity for **Viral Haemorrhagic Fevers** including Lassa fever and Ebola, assays for these pathogens are less available in other regions within the endemic region with samples being transferred to national, regional or international reference laboratories [@world_health_organisation_lassa_2022 ;@yadouleton_lassa_2020]. 

Conversely the majority of pathogens of animal species do not cause clinical disease in humans. Surveillance of pathogens in animal populations occurs for multiple reasons including animal health and welfare, conservation and agriculture. The information gathered by sampling efforts in animals can inform

```{r global-zoonotic-disease-surveillance, out.width = "100%", fig.cap = "Global Health Security Index country scores for the sub-domains of (top) 2.3) Real-time surveillance and reporting and (bottom) 1.2.2) Surveillance systems for zoonotic diseases/pathogens. Real time surveillance and reporting for epidemics of potential international concern is rated highly in several North and South American countries and countries in East and South East Asia and Oceania. Zoonotic disease surveillance is rated highly in European, North and South American countries and Oceania. Generally surveillance for zoonotic infectious disease is limited across much of Africa."}

ghs_data <- read_csv(here("thesis", "introduction_files", "ghs_index.csv"), show_col_types = FALSE)

ghs_data <- ghs_data %>%
  mutate(iso3c = countrycode(Country, origin = "country.name", destination = "iso3c")) %>%
  left_join(codelist %>%
              select(iso3c, un.region.name), by = "iso3c")

world <- ne_countries(scale = 110, returnclass = "sf") %>%
  select(iso_a3)

ghs_data %>%
  left_join(world, by = c("iso3c" = "iso_a3")) %>%
  st_as_sf() %>%
  ggplot() +
  geom_sf(aes(fill = Value)) +
  facet_wrap(~ Factor, ncol = 1) +
  scale_fill_viridis_c(option = "cividis") +
  labs(fill = "GHS Index score") +
  theme_bw()
```

### Predicting zoonotic spillover risk in a changing world

One purpose of surveillance in animal species is to inform risk prediction tools of outbreaks of known zoonotic infectious diseases and novel pathogen emergence. These tools aim to guide local public health responses through early warning systems or to effectively direct international investment towards pandemic prevention [@carlson_future_2021; @morse_prediction_2012]. Descriptions of previously reported zoonotic spillover events adjusting for reporting biases and combined with known host-pathogen distributions can highlight regions at increased risk [@jones_global_2008; @han_global_2016]. These models can also be used to identify host and pathogen species that require further investigations to understand pathogen prevalence. Bats (Chiroptera) and Rodents (Rodentia) are two taxa that contribute the greatest hazard of zoonotic spillover. These characteristics are driven by their widespread occurrence, encroachment of human activity within their natural habitats and species level traits that lead to high zoonotic pathogen burdens [references]. Pathogens that are predicted to spillover into human populations have more diverse pathogen characteristics and come from a wide range of viral, bacterial and fungal taxa. For this reason much of the prediction effort is focussed on the distribution of host species with pathogen prevalence assumed constant among much of a species range. There are several important examples that show violation of these simplifying assumptions. These examples typically come from host species with large home ranges but it is likely that this assumption does not hold true for most host-pathogen systems. For example *Lassa mammarenavirus* infection among its primary rodent host species has only been observed in its westernmost range, similarly for *Nipah henipavirus* which is observed only in the northern range of its primary bat host species (Figure \@ref(fig-lassa-nipah-range)).

```{r host-pathogen-associations, out.width = "100%", fig.cap = "The sampling of the global host-pathogen system is incomplete, and sparse. A recent effort to combine available data sources shows highlights the better understanding of pathogens of several mammal taxa. Focussing on Rodentia we can also observe temporal biases to pathogen identification."}

if(!file.exists(here("thesis", "introduction_files", "clover.rds"))) {
  clover <- list(bacteria = read_csv(gzcon(url("https://github.com/viralemergence/clover/raw/main/clover/clover_1.0_allpathogens/CLOVER_1.0_Bacteria_AssociationsFlatFile.csv")), show_col_types = FALSE),
                 other = read_csv(gzcon(url("https://github.com/viralemergence/clover/raw/main/clover/clover_1.0_allpathogens/CLOVER_1.0_HelminthProtozoaFungi_AssociationsFlatFile.csv")), show_col_types = FALSE),
                 viruses = read_csv(gzcon(url("https://github.com/viralemergence/clover/raw/main/clover/clover_1.0_allpathogens/CLOVER_1.0_Viruses_AssociationsFlatFile.csv")), show_col_types = FALSE)) %>%
    bind_rows()
  
  all_h_p <- clover %>%
    filter(HostClass == "mammalia") %>%
    filter(PathogenNCBIResolved == TRUE) %>%
    select(HostOrder, HostGenus, Host, PathogenType, Pathogen, ReleaseYear, PublicationYear) %>%
    mutate(year = pmin(ReleaseYear, PublicationYear, na.rm = TRUE)) %>%
    select(HostOrder, HostGenus, Host, PathogenType, Pathogen, year) %>%
    group_by(HostGenus, Host, PathogenType, Pathogen, year) %>%
    distinct() %>%
    group_by(HostOrder, HostGenus, Host, PathogenType, Pathogen) %>%
    arrange(year) %>%
    slice(1)
  
  summary_all <- all_h_p %>%
    distinct(HostOrder, Host, PathogenType, Pathogen) %>%
    group_by(HostOrder, Host, PathogenType, Pathogen) %>%
    summarise(N = n()) %>%
    group_by(HostOrder, PathogenType) %>%
    summarise(N = sum(N)) %>%
    arrange(-N)
  
  rodentia_clover <- all_h_p %>%
    filter(HostOrder == "rodentia")
  
  clover_data <- list(summary_all = summary_all,
                      rodentia_clover = rodentia_clover)
  
  write_rds(clover_data, here("thesis", "introduction_files", "clover.rds")) 
  
} else {
  
  clover_data <- read_rds(here("thesis", "introduction_files", "clover.rds"))
}

known_pathogens <- clover_data$summary_all %>%
  ungroup() %>%
  mutate(HostOrder = fct_inorder(str_to_sentence(HostOrder)),
         PathogenType = fct_inorder(str_to_title(str_replace_all(PathogenType, "/", " and ")))) %>%
  ggplot() +
  geom_col(aes(x = HostOrder, fill = PathogenType, y = N)) +
  scale_y_continuous(trans = "log10") +
  scale_fill_brewer(palette = "Dark2") +
  coord_flip() +
  facet_wrap(~ PathogenType) +
  theme_bw() +
  labs(x = "Host order",
       fill = "Pathogen type",
       y = "Number",
       title = "Known pathogens of mammal species")


rodent_pathogen_discovery <- clover_data$rodentia_clover %>%
  ungroup() %>%
  mutate(PathogenType = fct(str_to_title(str_replace_all(PathogenType, "/", " and ")), levels = levels(known_pathogens$data$PathogenType))) %>%
  ggplot() +
  geom_bar(aes(x = year, fill = PathogenType)) +
  scale_fill_brewer(palette = "Dark2") +
  facet_wrap(~ PathogenType, ncol = 1) +
  theme_bw() +
  guides(fill = "none") +
  labs(x = "Year",
       y = "Number",
       title = "Pathogen discovery in Rodentia")
  
known_pathogens

rodent_pathogen_discovery
```
```{r lassa-nipah-range, out.width = "100%", fig.cap = "Lassa mammarenavirus is an example of a rodent-borne zoonosis where pathogen prevalence is not consistent with host species range. Nipah henipavirus is an example of a bat-borne zoonosis where pathogen prevalance is not consistent with the host species range."}

```

### Rodent borne zoonotic infectious diseases

I have previously included examples that apply from multiple taxa of host species. For the remainder of this thesis I will focus on rodent borne zoonotic infectious diseases and will subsequently focus down on the case study of this thesis *Lassa mammarenavirus* in Sierra Leone.

The cause of this heterogeneity of pathogen prevalence and therefore spillover hazard within a hosts range is multifactorial. First, presence of additional microorganisms that are non-pathogenic to humans within a host species' range may provide cross immunity that prevent expansion of the zoonotic pathogen species into a wider area. Second, host species may be comprised of multiple clades which may have immunological differences which prevent efficient transmission of a pathogen adapted to one of the clades. Third, environmental suitability for the pathogen may vary across the host species range, this is of particular importance for pathogens that have environmental stages in the chain of transmission. Finally, presence of a pathogen in a host species may be dependent on the presence of other species or intermediate hosts for the pathogen that do not exist throughout the primary hosts range. Further, within a hosts range their occurrence and abundance may vary. For example in a species rich environment where a single host species conveys the hazard of spillover increased competition from conspecifics may reduce the host species' abundance in the landscape effectively "diluting" the hazard of spillover. Further, reduced biodiversity in a location may lead to non-host species not being present in a landscape, features of hosts that may contribute to their host status may also make them more resistant to factors that can drive local extinction and so these species are more likely to exist in species depauparate environments, increasing the hazard. Clearance of forest landscapes for monoculture agriculture may also lead to increased resources leading to increased populations of host species and increasing the scope for pathogen transmission among the species where previously this would not be sustained.

This heterogeneity will combine to modulate the hazard of zoonotic pathogen spillover from infected hosts into human populations. However, this is only a single layer of the risk of zoonotic pathogen spillover. The existence of the hazard in time and space alone will not necessarily lead to infection and disease in humans. This additional level is termed the risk and overlays the baseline hazard. Several factors may increase or decrease the risk of spillover. Human activity such as hunting for food may increase the risk of contact with an infectious host. Human activity in locations within close spatio-temporal windows as infected hosts may increase the risk of infection (i.e. using the same water sources). These also bring in societal levels of risk due to food security, access to clean water etc. Land use change may lead to infectious hosts accessing areas of human habitation or food storage as resources become less accessible in non-disturbed regions. Construction of human buildings in areas of habitation of the host species may lead to the host nesting in human domiciles for shelter. 


## Aims of the thesis 

The first aim of this thesis is to synthesise information on rodent and pathogen sampling from rodent trapping studies across West Africa to quantify the biases in currently available data. I hypothesise that rodent and pathogen sampling is spatially and taxonomically biased which will have implications on inference able to be drawn from currently available data about the hazard of zoonosis spillover risk across the region. I test the null hypothesis that rodent and pathogen sampling is conducted randomly in space across the region and propose an alternative hypothesis that rodent sampling is spatially clustered. I describe the occurrence of known and potential hosts of zoonosis from presence and absence data and compare this to currently available resources and produce host-pathogen associations from these data.


The second aim of this thesis is to investigate the association of rodent species diversity and landuse type in a Lassa fever endemic region of Eastern Sierra Leone. I hypothesise that known hosts of Lassa fever occur preferentially in human dominated landuse types with higher rodent species diversity in less disturbed landuse types. I test the null hypothesis that the probability of occurrence of rodent species does not change across landuse types and propose an alternative hypothesis that hosts of Lassa fever have a higher probability of occurrence in human dominated landuse types. 


The final aim of this thesis is to recreate Lassa fever transmission networks among rodent species. Using rodent trapping data

## Summary

\newpage