```{r introduction-setup, include=FALSE, echo = FALSE}
knitr::opts_chunk$set(echo = FALSE, dpi = 320, cache = TRUE,
                      message = FALSE,
                      fig.width = 8, fig.height = 8,
                      out.width = "80%", fig.align = 'center')
# packages
library(countrycode)
library(geodata)
library(here)
library(rnaturalearth)
library(sf)
library(tidyverse)
```


# Introduction {#introduction}


## Zoonotic infectious diseases


Zoonotic infectious diseases are diseases of humans caused by pathogens transmitted either directly (e.g., bites and scratches) or indirectly (e.g., via vectors, environmental or food contamination) from from animal hosts. Specifically, a “zoonoses” is any disease or infection that is shared between animals - including livestock, wildlife, and pets - and people, either through direct or indirect pathways  [@world_health_organization_taking_2019]. Zoonoses include bacterial, fungal, parasitic and viral microorganisms. The wider term “zoonotic disease” is often used for a disease that first originated in non-human animals and may continue to be used, even when disease transmission is no longer dependent on an animal reservoir (e.g., HIV, SARS-CoV-2) [@kock_situation_2022]. Individual transmission events from vertebrate animal populations into human populations - "spillover events" - can, lead to sustained outbreaks that may progress to localised epidemics or global pandemics [@plowright_pathways_2017]. Zoonotic pathogens do not always cause clinical disease in their animal hosts for example, *Lassa mammarenavirus* (LASV), the causative agent of Lassa fever in humans is not thought to cause significant clinical disease in rodent host species' as measured through organ dysfunction, weight loss or behavioural change [@safronetz_temporal_2022]. Meanwhile, in humans LASV infection can lead to severe clinical symptoms and death [@thielebein_virus_2022]. In contrast, Highly Pathogenic Avian Influenza, caused by *Influenza A virus* (subtype H5N1), leads to significant morbidity and mortality in infected bird species alongside pathogenicity in humans [@haider_unusually_2017; @writing_committee_of_the_second_world_health_organization_consultation_on_clinical_aspects_of_human_infection_with_avian_influenza_a_h5n1_virus_update_2008]. 


Zoonoses display a range of patterns of "spillover" from wild or domestic animals and transmission in human populations. Nipah virus infection (*Nipah henipavirus*) and LASV spillover events from wild animal sources occur at relatively frequent intervals but result in limited, onward human-to-human transmission leading to small-sized, geographically constrained outbreaks of human disease [@luby_recurrent_2009; @lo_iacono_using_2015]. Ebola virus disease (*Sudan ebolavirus* and *Zaire ebolavirus*) and mpox (formerly Monkeypox caused by the *Mpox virus*) in contrast exhibit sustained human-to-human transmission following spillover, but due to the transmission dynamics of these pathogens, outbreaks are generally constrained to local epidemics [@legrand_understanding_2007; @fine_transmission_1988]. Finally, some pathogens may be better adapted to transmission among humans due to their pathogen properties or similarities between human physiology or immunology and their primary vertebrate reservoir. These pathogens are able to rapidly expand beyond the geographic region of the initial spillover event via human transmission chains and may become zoonotic diseases with no further important transmission from wild or domestic animal populations (e.g., HIV and SARS-CoV-2) [@ye_zoonotic_2020; @marx_aids_2004]. Spillover may not be be limited to a single direction of animal to human transmission and "spillback" can play potentially important roles in maintaining pathogen endemicity with subsequent "secondary spillover" into human populations, alternatively, spillback can lead to morbidity and mortality in animal populations [@fagre_assessing_2022].


These different patterns of spillover are observable through phylogenetic analysis of viral sequences from human populations. Phylogenetic analysis of the SARS-CoV-2 virus suggests an initial spillover event into human populations in October and November of 2019 with establishment in the local human population ultimately leading to a global pandemic beginning in 2020 [@pekar_timing_2021]. Similarly, the multi-country mpox outbreak in 2022 is proposed to be secondary to human-to-human sustained transmission from a single origin endemic country, either directly linked to a spillover event or cryptic transmission among local human populations [@isidro_phylogenomic_2022]. In contrast, phylogenetic analysis of LASV sequences indicate the most common recent ancestor of viruses circulating in Nigeria is >1000 years prior, while sequences from Guinea and Sierra Leone suggest a more recent introduction of 220 and 150 years respectively [@andersen_clinical_2015]. These findings are consistent with repeated spillover events into human populations from pathogens circulating within a single or multiple reservoir species. While the 2022 mpox outbreak and ongoing SARS-CoV-2 pandemic are important examples of zoonoses causing epidemics and pandemics beyond their host species' ranges, these remain relatively rare events when compared to recurrent spillover events within endemic regions [@dudas_mers-cov_2018; @lloyd-smith_epidemic_2009]. The example of LASV highlights the risk of recurrent local spillover into human populations in endemic regions and reinforces the importance of surveillance of known zoonoses.


When considering interventions to reduce the health impact of zoonoses in endemic settings an approach that incorporates knowledge of multiple interacting systems are required. Understanding the role of environmental, wildlife and human factors on the hazard and risk of spillover events are necessary. This is often termed the "One Health" approach, a "collaborative, multisectoral, and transdisciplinary approach - working at the local, regional, national and global levels - with the goal of achieving optimal health outcomes recognizing the interconnection between people, animals, plants, and their shared environment." [@noauthor_one_2022]. This framework is particularly useful when considering how spillover of zoonoses occur in a context of ongoing climate, landuse and biodiversity change.


## Global change and zoonoses


Anthropogenic climate change is hypothesised to modify the risk of zoonoses to human populations through several mechanisms [@jones_zoonosis_2013; @daszak_anthropogenic_2001]. Changes in mean temperature and precipitation will alter environmental suitability for both pathogens and hosts leading to expansion or contraction of endemic regions [@mills_potential_2010]. Environmentally transmitted zoonoses such as Leptospira will become better able to persist in the environment under changes that increase ambient temperature in the presence of increased precipitation, leading to higher prevalence and incidence of infection [@lau_climate_2010; @llop_prediction_2022]. Vector borne zoonoses such as West Nile Virus are currently demonstrating range expansion as both mosquito vector abundance and occurrence is increased across a larger geographic range [@farooq_artificial_2022; @hoover_west_2016]. 

Climate change is occurring in step with anthropogenic landuse change. Human driven conversion of natural landscapes towards human dominated use occurs at both a local and global scale through direct and indirect human actions (i.e., agricultural development, natural resource extraction, and urbanisation) [@gottdenker_anthropogenic_2014]. The association of landuse change and pathogen transmission is complex, with increasing, decreasing and no change in pathogen transmission reported from observational studies of pathogen systems [@gottdenker_anthropogenic_2014]. Encroachment of human activity into zoonotic host animal ranges, as can occur under landuse change, has been hypothesised to increase the risk of spillover events into human populations, through increasing the animal-human interface raising the probability of direct and indirect contact with infected hosts of zoonoses [@murray_human_2013]. Additionally, heightened interactions between wildlife and domesticated animals may also increase the risk of subsequent zoonosis spillover into human populations where wild sylvatic animals are hosts of pathogens that can be amplified in domesticated animals (e.g., Nipah and Hendra virus) [@plowright_ecological_2015; @epstein_nipah_2006]. In tandem climate and landuse change also modify species' home ranges. As a consequence contact events between current hosts of zoonoses and potential future hosts of a pathogen are produced, increasing the potential for cross-species pathogen transmission and the subsequent expansion of a zoonoses' endemic range [@carlson_climate_2022]. This has been observed in Hendra virus where Southern range expansion of the black fruit bat (*Pteropus alecto*) has resulted in domesticated horses in Australia being infected, with subsequent spillover events into human populations [@yuen_hendra_2021].


Animal biodiversity also modulates zoonosis spillover risk, with several mechanisms proposed. The "Dilution effect" - initially applied to the Lyme disease (*Borrelia burgdorferi sensu lato*) system which comprises several vectors and animal hosts - hypothesises that in settings of low species diversity (measured as species richness) infection rates increase in a host species, the inverse being that higher levels of animal biodiversity reduces the rate of zoonosis spillover into human populations [@ostfeld_biodiversity_2000]. This theory has been supported by studies of several pathogen systems across parasites, bacteria, viruses and fungi [@keesing_impacts_2010; @civitello_biodiversity_2015]. There is ongoing debate as to whether this is a general property of zoonosis systems, as several studies have suggested the inverse. This mechanism, termed the "Amplification effect" occurs when increasing biodiversity, particularly through introduction of a new host, or a more competent host species can increase the rate of infection in hosts and potentially the risk of zoonosis spillover [@johnson_parasite_2012; @halliday_multivariate_2017]. These two effects may exist as a spectrum where dominance of one over the other is dependent on the specific disease context [@gomez-hernandez_concurrent_2023].


Climate, landuse and biodiversity change are interacting components within an ecosystem and attributing an effect of each independently to the risk of zoonosis spillover is challenging [@gibb_ecosystem_2020]. A synthesis of the effect of landuse change on biodiversity across multiple scales and zoonosis systems observed that species richness of zoonotic pathogen host species increased but not non-host species, along an anthropogenic landuse gradient [@gibb_zoonotic_2020]. These changes are also occurring at different rates globally. Climate, landuse and biodiversity change occurring in regions associated with a greater diversity of known zoonotic pathogens may potentially have a greater impact on the risk of zoonosis spillover than in settings of low diversity of zoonotic pathogens.


## Zoonoses discovery


The majority of microorganisms are non-pathogenic to humans or animals and provide vital ecosystem services and the small subset of microorganisms (<1%) that are pathogenic are typically able to replicate in multiple hosts [@editors_microbiology_2011; @woolhouse_diseases_2001; @woolhouse_population_2001]. For example, 60% of emerging human infectious diseases are associated with known zoonoses, a human infectious disease being a zoonoses is therefore not rare  [@jones_global_2008]. Discovery of these zoonoses is variable across mammalian taxa, with sampling effort increased in orders with increased human interaction, i.e., primates and livestock species. A recently compiled dataset (CLOVER), contains an increased number of described pathogens in Primates, Artiodactyla (ungulates) and Carnivora alongside Rodentia and Chiroptera [@gibb_data_2021; @gibb_viralemergenceclover_2021]. \@ref(fig:host-pathogen-associations) shows the number of known pathogens  in these mammalian orders. Of these, Rodentia contain the greatest number of pathogens known to be zoonotic [@han_global_2016]. 


```{r host-pathogen-associations, out.width = "100%", fig.cap = "The sampling of the global host-pathogen system is incomplete, and sparse. Bars indicate the number of known pathogens within different mammalian orders, the values within the bars indicate the number of species within the order known to host these pathogens. Data obtained from CLOVER."}

if(!file.exists(here("thesis", "introduction_files", "clover.rds"))) {
  
  bacteria <- read_csv(gzcon(url("https://github.com/viralemergence/clover/raw/main/clover/clover_1.0_allpathogens/CLOVER_1.0_Bacteria_AssociationsFlatFile.csv")), show_col_types = FALSE)
  other <-  read_csv(gzcon(url("https://github.com/viralemergence/clover/raw/main/clover/clover_1.0_allpathogens/CLOVER_1.0_HelminthProtozoaFungi_AssociationsFlatFile.csv")), show_col_types = FALSE)
  viruses <- read_csv(gzcon(url("https://github.com/viralemergence/clover/raw/main/clover/clover_1.0_allpathogens/CLOVER_1.0_Viruses_AssociationsFlatFile.csv")), show_col_types = FALSE)
                 
  clover <- list(bacteria = bacteria,
                 other = other,
                 viruses = viruses) %>%
    bind_rows()
  
  all_h_p <- clover %>%
    filter(HostClass == "mammalia") %>%
    filter(PathogenNCBIResolved == TRUE) %>%
    select(HostOrder, HostGenus, Host, PathogenType, Pathogen, ReleaseYear, PublicationYear) %>%
    mutate(year = pmin(ReleaseYear, PublicationYear, na.rm = TRUE)) %>%
    select(HostOrder, HostGenus, Host, PathogenType, Pathogen, year) %>%
    group_by(HostGenus, Host, PathogenType, Pathogen, year) %>%
    distinct() %>%
    group_by(HostOrder, HostGenus, Host, PathogenType, Pathogen) %>%
    arrange(year) %>%
    slice(1)
  
  summary_all <- all_h_p %>%
    distinct(HostOrder, Host, PathogenType, Pathogen) %>%
    group_by(HostOrder, Host, PathogenType, Pathogen) %>%
    summarise(N = n()) %>%
    group_by(HostOrder, PathogenType) %>%
    mutate(N_host = n_distinct(Host)) %>%
    group_by(HostOrder, PathogenType, N_host) %>%
    summarise(N = sum(N)) %>%
    arrange(-N)
  
  rodentia_clover <- all_h_p %>%
    filter(HostOrder == "rodentia")
  
  clover_data <- list(summary_all = summary_all,
                      rodentia_clover = rodentia_clover)
  
  write_rds(clover_data, here("thesis", "introduction_files", "clover.rds")) 
  
} else {
  
  clover_data <- read_rds(here("thesis", "introduction_files", "clover.rds"))
}

known_pathogens <- clover_data$summary_all %>%
  ungroup() %>%
  mutate(HostOrder = fct_inorder(str_to_sentence(HostOrder)),
         PathogenType = fct_inorder(str_to_sentence(str_replace_all(PathogenType, "/", " and ")))) %>%
  ggplot() +
  geom_col(aes(x = HostOrder, fill = PathogenType, y = N)) +
  geom_label(aes(x = HostOrder, y = N_host, label = N_host)) +
  scale_y_continuous(trans = "log10") +
  scale_fill_brewer(palette = "Dark2") +
  coord_flip() +
  facet_wrap(~ PathogenType) +
  theme_bw() +
  labs(x = "Order",
       fill = "Pathogen group",
       y = "Number of pathogens",
       title = element_blank())


known_pathogens
```


The discovery of zoonoses are biased by both our ability to detect them and the sampling effort of animal species for pathogens [@grange_ranking_2021; @gibb_mammal_2021]. The discovery rate of viral zoonoses, an important subset of all zoonoses, has increased with improvements in the technical means to identify them [@woolhouse_temporal_2008]. The rate of discovery has exceeded prior expectations of viral biodiversity, but, continues to remain taxonomically and geographically biased, limiting inference able to be drawn from current datasources [@wille_how_2021]. Similar limitations are likely for other zoonoses taxa including bacteria, fungi and parasites. The general trend of increasing rates of pathogen discovery over time are shown for Rodentia in \@ref(fig:rodent-pathogen-discovery).

```{r rodent-pathogen-discovery, out.width = "100%", fig.cap = "Discovery of pathogens in Rodentia, the order containing the greatest number of zoonotic pathogens, has increased over time. Data obtained from CLOVER."}

rodent_pathogen_discovery <- clover_data$rodentia_clover %>%
  ungroup() %>%
  mutate(PathogenType = fct(str_to_sentence(str_replace_all(PathogenType, "/", " and ")), levels = levels(known_pathogens$data$PathogenType))) %>%
  ggplot() +
  geom_bar(aes(x = year, fill = PathogenType)) +
  scale_fill_brewer(palette = "Dark2") +
  facet_wrap(~ PathogenType, ncol = 1) +
  theme_bw() +
  guides(fill = "none") +
  labs(x = "Year",
       y = "Number of pathogens",
       title = element_blank())

rodent_pathogen_discovery
```


Zoonoses are known to exist in the majority of terrestrial mammal orders (21/27) with the number of hosts of zoonotic pathogens stronglyt positively associated with the species richness of these orders [@han_global_2016]. Two mammalian taxa, Rodentia and Chiroptera are associated with the greatest number of species that are hosts of zoonoses and overall number of zoonoses [@han_rodent_2015]. It is unclear whether these taxa represent special reservoirs that lead to an increased proportion of zoonotic viruses circulating within these species or make them more likely to transmit pathogens to humans, or whether the increased number of zoonoses associated with these taxa is driven by their increased species richness [@olival_are_2012; @wolfe_origins_2007; @luis_comparison_2013; @mollentze_viral_2020]. Geographic hotspots of zoonotic disease risk are therefore predicted to occur where mammalian host species richness is greatest, e.g., in the tropics [@han_global_2016].


West Africa is one such location of high mammalian biodiversity [@ceballos_global_2006]. This region is also undergoing significant anthropogenic change, driven by increasing human populations, agricultural development, urbanisation and resource extraction alongside effects of anthropogenic climate change such as desertification and changes in precipitation dynamics [@haggblade_regulating_2022; @maconachie_diamond_2012; @walther_urbanisation_2021; @nicholson_desertification_1998; @bongaarts_human_2009]. It has also been the location of several recent zoonosis epidemics and outbreaks, for example, the 2014 Ebola epidemic and ongoing Lassa fever outbreaks. 


## West Africa as a hotspot of zoonosis risk


While the number of zoonotic infectious disease outbreaks and, human morbidity and mortality associated with them, has been observed to rise in West Africa this must be viewed in the local context of anthropogenic change described above, particularly as the number of people at risk of infection is continuing to increase [@makoni_africas_2020]. Alongside these global changes pathogen discovery, improved access to diagnostics, increasing healthcare access and improved reporting will together result in an apparent increase in the burden of zoonotic infectious diseases in the region. An example of intensifying pathogen discovery is the PREDICT programme, conducted between 2009 and 2020, it tested in excess of 164,000 samples from animals and humans in 14 African countries and 12 Asian countries identifying 949 novel viruses including 217 known zoonoses including the detection of Marburg virus for the first time in Sierra Leone, West Africa [@noauthor_about_2019; @amman_isolation_2020]. Projects such as this can importantly change our understanding of the locations of zoonoses, although these pathogens have likely circulated in the region for many years prior to discovery. Improved diagnostics and reporting of zoonoses are evident in the case of Lassa fever, particularly in Nigeria. Here, the Nigerian Center for Disease Control (NCDC) have expanded the availability of testing for this disease. Prior to 2005 molecular diagnosis of Lassa fever infection was not possible in the country with samples transferred to the Lassa fever unit at Kenema General Hospital, Sierra Leone [@naidoo_nigerias_2020]. Between 2005 and 2012 testing was established in Lagos and Irrua, Nigeria with further laboratory capacity established at the National Reference Laboratory in Abuja and in Ebonyi state in 2018. The expansion of testing capacity has led to in excess of 20,000 individuals being tested for Lassa fever between 2018 and 2021, any trends in the number of reported cases of this disease from Nigeria need to be considered in light of this [@dalhat_epidemiological_2022].


Current detection of zoonotic infectious disease outbreaks generally rely upon clinical case detection of infected humans within healthcare settings (i.e., real-time surveillance and reporting) rather than monitoring transmission among wild or domestic animals (i.e., zoonotic disease surveillance). No public health systems have implemented active surveillance systems through testing of animal populations in West Africa. Elsewhere, for example in Europe, active srveillance in birds and horses is conducted for West Nile Virus to inform risk assessments of human disease outbreaks [@gossner_west_2017]. The Global Health Security Index measured activities conducted by countries to assess their ability to respond to an emerging outbreak of a zoonotic infectious disease [@global_health_security_index_ghs_2022]. Figure \@ref(fig:global-zoonotic-disease-surveillance) shows results from two components of this assessment, highlighting that few African countries have widely implemented real-time surveillance or zoonotic disease surveillance in animals. Real-time surveillance is rated as poor, with the notable exception of Nigeria, suggesting that these countries may not be able to rapidly identify outbreaks of endemic zoonotic diseases of epidemic potential (i.e., Ebola, mpox and Lassa fever). Zoonotic disease surveillance among animal host species in West Africa is currently limited to academic or programmatic research which informs local policy and identifies regions at potentially greater risk for spillover events. This information has been used by public health agencies to aid risk stratification of patients that present with symptoms consistent with these diseases, based on when, where, and why they present to local healthcare services [@leski_sequence_2015; @happi_increased_2022]. Few countries globally, with none in West Africa, have surveillance systems that combine animal and human data [@wendt_zoonotic_2015].

```{r global-zoonotic-disease-surveillance, out.width = "100%", fig.cap = "Global Health Security Index country scores for the sub-domains of2.3) Real-time surveillance and reporting  (top) and 1.2.2) Zoonotic disease surveillance (bottom). Real-time surveillance and reporting for epidemics of potential international concern is rated highly in several North and South American countries and countries in East and South East Asia and Oceania. Zoonotic disease surveillance in animals is rated highly in European, North and South American countries and Oceania. Generally surveillance for zoonotic infectious disease is limited across much of Africa, with the notable exception of Nigeria for real-time surveillance and reporting. Data obtained from the Global Health Security Index."}

ghs_data <- read_csv(here("thesis", "introduction_files", "ghs_index.csv"), show_col_types = FALSE)

ghs_data <- ghs_data %>%
  mutate(iso3c = countrycode(Country, origin = "country.name", destination = "iso3c")) %>%
  left_join(codelist %>%
              select(iso3c, un.region.name), by = "iso3c")

world <- ne_countries(scale = 110, returnclass = "sf") %>%
  select(adm0_a3, admin)

ghs_data %>%
  left_join(world, by = c("iso3c" = "adm0_a3")) %>%
  st_as_sf() %>%
  ggplot() +
  geom_sf(aes(fill = Value)) +
  facet_wrap(~ Factor, ncol = 1) +
  scale_fill_viridis_c(option = "cividis", direction = -1) +
  labs(fill = "GHS Index score") +
  theme_bw()
```



## Rodent borne zoonoses

Rodents are a diverse, globally distributed mammalian order that provide important and beneficial ecosystem services including pest regulation and seed dispersal [@fischer_ecosystem_2018]. Of the almost 2,600 species, representing 40% of all mammalian species, 282 species (~11%) have been identified to be reservoirs of 95 known zoonoses, a greater number than other mammal orders [@delia_rodent_2019; @ecke_population_2022; @han_global_2016]. The majority of these zoonoses are viruses (34) and bacteria (26) with the remaining including helminths, protozoa and fungi. As discussed above the high prevalence in this order may be driven by high species richness, rather than any inherent properties of the order Rodentia [@mollentze_viral_2020]. 


Within this order, the prevalence of zoonoses are disproportionally high within species that demonstrate "fast" life history strategies, although the effect of sampling biases and confounding effects such as synanthropy may be producing some of this observed effect [@han_rodent_2015; @albery_fast-lived_2021]. Fast-lived rodent species (i.e., those prioritising reproduction over survival and longevity), are typically small, abundant and are more commonly urban-adapted [@albery_fast-lived_2021]. These species favour inexpensive, nonspecific immune defenses, which make them more likely to be hosts of zoonoses, although whether these properties are consistent within genera is unclear and whether these findings are replicated in wild, as opposed to laboratory, animals is unknown [@martin_immune_2007; @viney_immunology_2017].


Irrespective of the causal drivers of high zoonoses prevalence among rodent species, components of their life histories increase the risk of spillover into human populations. Synanthropy describes an organism that lives near and benefits from humans and their environmental modifications, this property is common among rodent species, more so among rodent species known to be reservoirs of zoonoses [@ecke_population_2022]. Synanthropic species tend to be highly abundant in locations in which they occur, with high population densities and dynamic population fluctuations in response to resource availability, which promotes both fequency- and density-dependent transmission of pathogens among hosts [@ecke_population_2022]. The high abundance of these species in human dominated landscapes increases the rate of contact with humans providing increased opportunities for both direct- and indirect transmission of rodent borne zoonoses [@morand_changing_2019; @iacono_unified_2016].


Rodent species that have wide ranges may display heterogeneity across their range in both their biology and behaviour. For example, studies in *Clethrionomys* voles, hosts of Puumala orthohantavirus, have been observed to display different population dynamics across a latitudinal gradient from Northern Finland to Central Europe, affecting pathogen dynamics within these populations [@turchin_empirically_1997; @henttonen_small_2001]. Similarly, while the primary reservoir species of LASV, *Mastomys natalensis*, has been observed to have dramatic population fluctuations in the Eastern extent of its range (Tanzania), the same amplitude of population fluctuations have not been observed in West African populations (i.e., Guinea) where they host LASV [@fichet-calvet_reproductive_2008; @leirs_stochastic_1997].  This may impact the generalisability of studies conducted in within a rodents range when attempting to understand the risk of rodent borne zoonosis spillover.


## Lassa fever: A case study of a rodent borne zoonosis in West Africa


The above sections have introduced zoonotic infectious diseases, the effect of a changing world on potential disease emergence, the particular risk of emergence and outbreaks in West Africa and the role of rodents in zoonotic infectious disease transmission. The remainder of this introduction will focus on the case study of this thesis, Lassa fever, in West Africa and more specifically Sierra Leone.


### *Lassa mammarenavirus* and Lassa fever


*Lassa mammarenavirus* an enveloped, bisegmented, single stranded RNA virus of the Arenaviridae family is a zoonotic pathogen and is the causative agent of Lassa fever in humans. Lassa fever is a potentially lethal viral haemorrhagic fever, first identified from a case series of infected patients seeking healthcare in Jos, Nigeria in 1969,  [@frame_lassa_1970]. Human infection is caused by spillover of the virus from infected rodents and their excreta, with a limited role of human-to-human secondary transmission [@mccormick_prospective_1987; @lo_iacono_using_2015]. The primary host of LASV has been identified as the multimammate rat (*M. natalensis*) following an outbreak in Sierra Leone between 1970-2 [@monath_lassa_1974]. This synanthropic rodent species is found across much of sub-Saharan Africa, however, outside of West Africa no individuals of this species have been found to be infected with this virus [@grobbelaar_mammarenaviruses_2021; @bellocq_dhati_2020; @colangelo_mitochondrial_2013].


*Lassa mammarenavirus* has four confirmed lineages (I-IV) and three additional lineages (V-VII) based on geographic and phylogenetic analysis [@li_genetic_2023]. Lineages I, II, III and VI are located within Nigeria, lineage IV contains all isolates from the Mano River region of Guinea, Liberia and Sierra Leone, lineage V contains samples from Mali and Ivory Coast and lineage VII contains recently sampled sequences from Togo [@ehichioya_phylogeography_2019; @andersen_clinical_2015; @manning_lassa_2015; @whitmer_new_2018]. Lineage I is believed to be the most ancient, originating around 1,000 years ago in the North East of Nigeria, with subsequent radiation and establishment of lineages II and III in the Southern and Central areas of the country respectively [@ehichioya_phylogeography_2019; @andersen_clinical_2015]. Lineage IV represents a Westward expansion of the virus into the Mano River region, dated around 350 years ago [@andersen_clinical_2015].


Host cell entry of the virus is mediated by a trimeric glycoprotein complex that interacts with host cell receptors and leads to fusion of the viral and host membranes, *in vivo* this protein undergoes substantial host-derived glycosylation, effectively reducing available antibody binding domains [@hastie_lassa_2018]. Once within the host cell the viral nucleoprotein associates with viral RNAs forming ribonucleoprotein complexes facilitating transcription and replication of viral RNA within the host cell cytoplasm [@hass_replicon_2004]. The process of viral entry into host cells is expected to lead to the observed tissue tropism in experimental infection models in guinea pigs and *M. natalensis* [@torriani_lassa_2017]. Within infected guinea pigs and *M. natalensis* LASV load was highest transiently in the lymph nodes with sustained high titres in the lungs and spleen [@jahrling_pathogenesis_1982; @safronetz_temporal_2022]. Minimal pathological changes were observed in guinea pigs or *M. natalensis*, with no evidence of clinical disease in these animals.


Among infected humans with clinical symptoms the viral incubation period is between 7 and 18 days [@mccormick_prospective_1987]. Initial symptoms are non-specific with fever, weakness, malaise, cough, sore throat and a typically frontal headache [@knobloch_clinical_1980]. The majority of symptomatic patients will go on to develop joint and lumbar pain, a non-productive cough with many developing severe retrosternal chest pain, nausea with vomiting and diarrhoea and abdominal pain [@mccormick_lassa_2002]. Up to a third of hospitalised patients will significantly decline 6-8 days post onset of fever with a minority developing haemorrhagic syndome with bleeding from the mucosal surfaces. Severe pulmonary oedema and soft tissue oedematous changes in the head and neck are common in fatal cases [@knobloch_clinical_1980]. The vast majority of infections, commonly reported as 80%, are asymptomatic although in the absence of long term prospective studies the proportion of asymptomatic infections is difficult to estimate [@mccormick_prospective_1987]. There is some limited evidence that disease severity may vary by infecting lineage [@garry_lassa_2023].


Treatment options for acute cases of Lassa fever are limited. Ribavirin is the standard of care for treating acute cases although the effectiveness of this treatment is questionnable [@salam_ribavirin_2022]. Supportive care therefore remains the mainstay of treatment for hospitalised individuals. There are no currently available vaccinations for Lassa fever, although three candidate vaccines have begun clinical trials [@salami_review_2019; @inovio_pharmaceuticals_dose-ranging_2022; @inovio_pharmaceuticals_study_2020; @international_aids_vaccine_initiative_phase_2023; @themis_bioscience_gmbh_randomized_2022].


The case-fatality rate of Lassa fever has been reported to be as high as 29.7% although this varies by country and year [@kenmoe_systematic_2020]. This estimate is based on a systematic review of the published scientific literature and does not include data from epidemiological reports or WHO bulletins. I conducted a review of both epidemiological reports and the published literature to derive the case-fatality rate among confirmed cases in order to estimate the scale of underreporting in Lassa fever producing an estimated case-fatality rate of 16.5% (+/- 5%) among confirmed cases [@simons_lassa_2022]. Importantly this estimate is sensitive to biases in reporting and is likely a grossly inflated rate of mortality. Severe cases are more likely to come into contact with healthcare services and be tested for Lassa fever, these cases are also more likely to result in disease associated mortality skewing confirmed cases to those with severe disease. Therefore, this case-fatality rate should be considered a severe disease case-fatality rate, the majority of mildly symptomatic cases will have a dramatically reduced probability of mortality and will lower the case-fatality rate.


Survivors of symptomatic Lassa fever may have lasting effects of the disease. Sensorineural hearing loss is reported to occur in up to a third of Lassa fever survivors and potentially causes significant social and public health burden in the region that have not been well studied [@mateer_lassa_2018]. Additional neurological sequealae reported in Lassa fever survivors include cerebellar ataxia and visual impairment, although few patients have been assessed for these complications and progression over time is unclear [@li_ophthalmic_2020; @ezeomah_sequelae_2019]. Most hospitalised patients, following recovery, rapidly clear viral RNA. Most patient sera is negative for viral RNA at hospital discharge, however, up to 50% of male survivors have detectable viral loads in seminal fluid at 3 months post-hospitalisation raising concerns that human-to-human sexual transmission may be possible in this cohort [@thielebein_virus_2022].


### Lassa fever epidemiology


Annual Lassa fever incidence is unknown, with estimates ranging between 150,000 to 4,300,000 cases per year cases annually [@mccormick_prospective_1987; @basinski_bridging_2021]. The wide uncertainty surrounding these estimates is due to a combination of few serological studies, limited disease surveillance and an overlap between the symptomatology of Lassa fever with other infectious diseases in these endemic regions (e.g., malaria). Lassa fever is currently considered endemic in 8 West African countries: Benin, Ghana, Guinea, Liberia, Mali, Nigeria, Sierra Leone and Togo by the World Health Organisation (WHO) [@world_health_organisation_lassa_2022]. Sporadic cases in Burkina Faso and Ivory Coast have also been reported (\@ref(fig:lassa-endemic)). The endemic region is consistent with the expected range of the primary reservoir species *M. natalensis*. Imported cases have been reported from non-West African countries such as the United Kingdom, Germany and the United States of America with few observed events of secondary human-to-human transmission outside of the endemic region [@tuite_potential_2019; @wolf_fifty_2020].


```{r lassa-endemic, warning = FALSE, message = FALSE, echo = FALSE, out.width = "100%", fig.cap = "Lassa fever is considered endemic in eight West African countries, sporadic outbreaks have been reported from a further two countries within the region. The red border indicates the range of *Mastomys natalensis* in West Africa, it's range extends East and South across the continent (not shown here). Data on Lassa fever endemicity is obtained from the WHO, data on *Mastomys natalensis* range is obtained from the International Union for Conservation of Nature Red List."}

continental_wa <- c("BEN", "BFA", "CIV", "GHA", "GIN", "GMB", "GNB", "LBR", "MLI", "NER", "NGA", "SEN", "SLE", "TGO")
endemic_lassa <- c("BEN", "GHA", "GIN", "LBR", "MLI", "NGA", "SLE", "TGO")
outbreak_lassa <- c("BFA", "CIV")

# Download shapefiles from GADM and save them
wa_shape <- gadm(country = continental_wa, level = 0, path = here("thesis", "introduction_files", "wa_gadm"), version = "latest") %>%
  st_as_sf() %>%
  select(GID_0, COUNTRY)

wa_region <- wa_shape %>%
  filter(GID_0 %in% continental_wa) %>%
  mutate(status = factor(case_when(GID_0 %in% endemic_lassa ~ "Endemic",
                            GID_0 %in% outbreak_lassa ~ "Outbreaks",
                            TRUE ~ "No cases"), levels = c("Endemic", "Outbreaks", "No cases")))

wa_region$COUNTRY[wa_region$COUNTRY == "Côte d'Ivoire"] <- "Ivory Coast"

mastomys_range <- read_rds(here("thesis", "introduction_files", "mastomys_iucn.rds")) %>%
  st_crop(st_bbox(wa_region))

ggplot() +
  geom_sf(data = wa_region, aes(fill = status)) +
  geom_sf(data = mastomys_range, colour = "#C12D20", lwd = 2, fill  = NA) +
  geom_sf_label(data = wa_region, aes(label = COUNTRY)) +
  scale_fill_viridis_d(option = "cividis") +
  theme_bw() +
  labs(x = element_blank(),
       y = element_blank(),
       fill = "Lassa endemicity")


```


Nigeria and Sierra Leone have historically reported the greatest number of Lassa fever cases (\@ref(fig:lassa-cases)). Potentially this is driven by increased availability of testing for acute cases in these countries. Human seroepidemiological surveys in Guinea, Mali and Ivory Coast, countries that have generally reported few acute cases, report seroprevalence in excess of 20% suggestive of undetected localised transmission of Lassa fever [@kerneis_prevalence_2009; @bausch_lassa_2001; @sogoba_lassa_2016; @safronetz_annual_2017; @akoua-koffi_detection_2006]. The number of reported cases across the region declined during the Ebola and SARS-CoV-2 epidemic where changes in healthcare seeking behaviour and availability of Lassa fever testing may have reduced. The number of cases reported in Nigeria has generally increased since data became routinely available, in contrast there has been a dramatic fall in cases reported from Sierra Leone, whether these represent changes in the underlying spillover risk is currently unclear.


```{r lassa-cases, warning = FALSE, message = FALSE, echo = FALSE, out.width = "100%", fig.cap = "Confirmed Lassa fever cases from countries in West Africa 2008-2023. Confirmed cases show variability by year with the greatest number of cases reported from Nigeria, Sierra Leone and Liberia. Grey shaded regions represent periods of regional or global epidemics which may have affected Lassa fever reporting (i.e., the Ebola epidemic and SARS-CoV-2 pandemic). The yellow shaded region represents 2023 where an incomplete year is shown. Data compiled from multiple sources."}

lassa_raw <- read_csv(here("thesis", "introduction_files", "confirmed_cases.csv"))

lassa_clean <- lassa_raw %>%
  group_by(year, country) %>%
  mutate(n_records = n()) %>%
  ungroup() %>%
  filter(!(country == "nigeria" & region == "all" & n_records > 1)) %>%
  mutate(country = str_to_title(str_replace_all(country, "cote d'ivoire", "ivory coast")),
         GID_0 = countrycode(sourcevar = country, origin = "country.name", destination = "iso3c"),
         status = factor(case_when(GID_0 %in% endemic_lassa ~ "Endemic",
                                   GID_0 %in% outbreak_lassa ~ "Outbreaks",
                                   TRUE ~ "No cases"), levels = c("Endemic", "Outbreaks", "No cases")),
         country_label = factor(case_when(country %in% c("Benin", "Ghana", "Guinea", "Liberia", "Mali",
                                                         "Nigeria", "Sierra Leone", "Togo") ~ country,
                                          TRUE ~ "Combined"),
                                levels = c("Benin", "Ghana", "Guinea", "Liberia", "Mali",
                                           "Nigeria", "Sierra Leone", "Togo", "Combined"))) %>%
  group_by(year, country, country_label, status) %>%
  summarise(n_cases = sum(confirmed_cases, na.rm = TRUE))

colours_wa <- c("Endemic countries" = "white",
                "Benin" = "#871a10",
                "Ghana" = "#000000",
                "Guinea" = "#be2a2c",
                "Liberia" = "#0a2864",
                "Mali" = "#50ab4a",
                "Nigeria" = "#398053",
                "Sierra Leone" = "#2f71c1",
                "Togo" = "#f7d147",
                " " = "white",
                "Non-endemic countries" = "white",
                "Ivory Coast" = "#e6853b",
                "Burkina Faso" = "#107d87")

lassa_clean$country = factor(lassa_clean$country, levels=c("Endemic countries", "Benin", "Ghana", "Guinea",
                                                           "Liberia", "Mali", "Nigeria", "Sierra Leone", "Togo", " ",
                                                           "Non-endemic countries", "Ivory Coast", "Burkina Faso"))

ggplot(lassa_clean, aes(x = year, y = n_cases, fill = country)) +
  geom_col() +
  scale_x_continuous(breaks = seq(from = min(lassa_clean$year), to = max(lassa_clean$year), by = 4)) +
  scale_y_continuous(breaks = function(x) unique(floor(pretty(seq(0, (max(x) + 1) * 1.1))))) +
  scale_fill_manual(values = colours_wa,
                    drop = FALSE) +
  guides(fill=guide_legend(ncol = 1)) +
  theme(legend.key = element_rect(fill = NA),
        legend.title = element_blank()) +
  facet_wrap(~ status + country_label, scales = "free_y") +
  annotate(geom = "rect", xmin = 2013, xmax = 2016, ymin = -Inf, ymax = Inf, alpha = 0.1) +
  annotate(geom = "rect", xmin = 2020, xmax = year(Sys.Date()), ymin = -Inf, ymax = Inf, alpha = 0.1) +
  annotate(geom = "rect", xmin = year(Sys.Date()), xmax = year(Sys.Date()) + 1, ymin = -Inf, ymax = Inf, alpha = 0.2, fill = "orange") +
  theme_minimal() +
  labs(x = element_blank(),
       y = "Confirmed cases",
       fill = element_blank())
```


The number of reported confirmed cases of Lassa fever in endemic countries is likely significantly underreported. Cases tend to occur in rural and remote locations where healthcare access is generally low, and financial and societal costs of accessing healthcare relatively high [@bhadelia_understanding_2019; @nnaji_deuce-ace_2021]. Additionally while clinicians in endemic settings have good awareness of symptoms that may indicate acute Lassa fever infection, access to testing and timely reporting were identified as factors that could lead to diagnostic delay, poor patient outcomes and delayed public health responses to outbreaks [@olowookere_diagnostic_2014; @rohan_beyond_2022]. An estimate of the degree of underreporting was conducted using reported Lassa fever disease associated mortality, assuming a consistent 16.5% case-fatality rate across the region. Using this approach Nigeria was found to report the highest proportion of all expected cases (63%) while countries with generally fewer observed outbreaks reported significantly fewer than expected cases (e.g., Ghana - 17%, Guinea - 25%) [@simons_lassa_2022].

Human seroepidemiological studies conducted in several regions of Sierra Leone suggest that despite the observed fall in human cases of disease infection remains prevalent [@grant_seroprevalence_2023].This study in Sierra Leone is also suggestive that widespread transmission of LASV is occurring outside the traditionally considered endemic region of Eastern Sierra Leone. A large scale serological study conducted by the Coalition for Epidemic Preparedness Innovations across Benin, Guinea, Liberia, Nigeria and Sierra Leone to understand the prevalence to antibodies against LASV has been implemented and results are awaited [@penfold_prospective_2023].


### Rodent hosts of *Lassa mammarenavirus*


While *M. natalensis* is considered the primary reservoir of LASV 11 other rodent species have been found to be acutely infected or have antibodies to the virus [@simons_rodent_2023]. The role of the wider rodent species community in viral transmission in endemic areas is not currently well understood. Further, evidence exists for prior exposure to LASV in non-rodent species, including domestic dogs, non-human primates and shrews the role of these species is even less clear [@kenmoe_systematic_2020].


*Mastomys natalensis* is a synanthropic rodent species, native to Africa. This species is considered a pest species across much of its range, as it lives within and around human communities consuming grain within the fields and in stores [@swanepoel_systematic_2017]. The species demonstrates archetypal fast life history traits with rapid sexual maturity (4 months), short life span (<1 year) and large litter sizes (mean of 9 live offspring) [@coetzee_biology_1975; @albery_fast-lived_2021; @safronetz_establishment_2021]. The proportion of reproductively active individuals have been observed to increase in the late wet season and early dry season with a nadir in the late dry season, leading to a population boom in the late wet season [@mlyashimbi_relationships_2018; @mayamba_population_2021]. Importantly the majority of population dynamic studies in this species have been conducted in Tanzania where population dynamics have been observed to be closely linked to food availability, the same drivers of these population dynamics may not be as extreme in West Africa where the population of this species has been observed to be less closely linked to rainfall patterns [@olayemi_small_2018; @fichet-calvet_reproductive_2008; @bangura_lassa_2021]. 


As a synanthropic species *M. natalensis* typically occurs within areas of human habitation and agriculture and is found to be an early invader of land converted to agricultural use [@makundi_reproduction_2007]. This landuse preference is consistent across the entirety of its range with few individuals trapped in forested landscapes [@olayemi_small_2018; @fichet-calvet_reproductive_2008; @bangura_lassa_2021; @coetzee_biology_1975; @leirs_spatial_1996]. This would suggest that abundance of this species is heterogeneous across the it's proposed range with expected absence in the forested regions of sub-Saharan Africa. This species is considered non-territorial, with a limited home range of around 30m, although dispersal across greater distances has been observed [@leirs_spatial_1996]. Contact with other rodent species is therefore assumed to be common and this is reflected by the high rodent species richness in locations where *M. natalensis* is detected [@fichet-calvet_reproductive_2008; @bangura_lassa_2021]. This may be potentially important for the transmission of LASV among rodent hosts and facilitate transmission of the virus across the heterogeneous landuse types of the endemic region.


While this species is distributed across sub-Saharan Africa genomic studies suggest that six phylogroups (A-I to A-III and B-IV to B-VI) have formed which correspond to different geographic regions of Africa. The West African clade, A-I is genetically distinct and is found in the endemic region of LASV [@colangelo_mitochondrial_2013]. Individuals of the other clades have not been found to be infected with LASV but have been identified to be infected with other Arenaviridae, including Mayo Ranewo (A-II), Dhati Welel (A-III), Gairo (B-IV), Morogoro (B-V), and Mopeia viruses (B-VI) [@bellocq_dhati_2020]. The presence of clade specific Arenaviridae may explain why the Lassa fever endemic region is constrained to the Western radiation of *M. natalensis* despite it being found throughout sub-Saharan Africa and may limit any future expansion of the virus.


While LASV is considered to primarily infect *M. natalensis* the prevalence of LASV in rodent communities varies importantly across the region and over time. \@ref(fig:rodent-lassa-seroprevalence) shows the prevalence of acute infection or antibodies to LASV among sampled *M. natalensis* communities. When detected LASV prevalence varies both within and between countries, in Guinea LASV was detected in 10 to 55% of trapped individual rodents, in a single study conducted in Mali acute infection was detected in 25% of individuals, Sierra Leone showed a wide range of prevalence from 5% to 100%, most of the sampling events in these settings did not detect any acute infection (not shown here). The number of individuals tested for acute infection is typically lower than those tested for antibodies due to availability of reagents, cost and laboratory requirements. There are also selection biases in which rodents are tested for acute infection which may increase the proportion of positive samples, for example, testing may only be performed in antibody positive rodents or rodents trapped in the location of a confirmed human case. 


```{r rodent-lassa-seroprevalence, warning = FALSE, message = FALSE, echo = FALSE, out.width = "100%", fig.cap = "Prevalence of acute infection with LASV or antibodies to LASV in rodent species sampled in West Africa. The size of a point relates to the number of samples of that species tested and the colour to the country in which the rodent was sampled. Where possible the rodent species is identified, for individuals only identified to genus level the genera from which samples obtained is shown. Six rodent species have been found to be acutely infected with LASV with 10 species having detectable antibodies. The majority of samples have been obtained from rodents trapped in Guinea and Sierra Leone. Data obtained from ..."}

review_data <- read_tsv(here("thesis", "introduction_files", "taxa.txt")) %>%
  filter(str_detect(specificEpithet, "Lassa") & occurrenceStatus == "Present")

lassa_testing <- review_data %>%
  mutate(rodent_species = str_remove_all(associatedTaxa, "parasite of "),
         n_tested = as.numeric(str_extract(occurrenceRemarks, "\\d+")),
         n_positive = as.numeric(organismQuantity),
         method = case_when(str_detect(identificationRemarks, "PCR|Culture") ~ "Acute infection",
                            str_detect(identificationRemarks, "Serology") ~ "Serology"),
         rodent_species = fct_rev(factor(str_to_sentence(case_when(rodent_species == "mastomys" ~ "mastomys spp.",
                                                           rodent_species == "crocidura" ~ "crocidura spp.",
                                                           rodent_species == "praomys" ~ "praomys spp.",
                                                           rodent_species == "rattus" ~ "rattus spp.",
                                                           rodent_species == "rodentia" ~ "rodentia spp.",
                                                           TRUE ~ rodent_species)),
                                 levels = c("Mastomys natalensis", "Mastomys erythroleucus", "Mastomys spp.", "Rattus rattus", "Rattus spp.", "Praomys daltoni", "Praomys rostratus", "Praomys spp.", "Gerbilliscus kempii", "Mus minutoides", "Mus baoulei", "Mus musculus", "Hylomyscus pamfi", "Lemniscomys striatus", "Crocidura spp.", "Lophuromys sikapusi", "Rodentia spp."))),
         perc_positive = round(n_positive/n_tested * 100, 1)) %>%
  select(country, rodent_species, perc_positive, n_tested, n_positive, method, bibliographicCitation)

lassa_testing$perc_positive[lassa_testing$perc_positive == 200] <- 100

ggplot() +
  geom_point(data = lassa_testing, aes(y = rodent_species, x = perc_positive, size = n_tested, colour = country), position = position_jitter(height = 0.2), alpha = 0.5) +
  scale_size(range = c(1, 6)) +
  scale_colour_brewer(palette = "Set1") +
  facet_grid(~ method) +
  theme_bw() +
  labs(y = element_blank(),
       x = "Percentage positive",
       colour = "Country",
       size = "Number assayed")

```


\@ref(fig:rodent-lassa-seroprevalence) also highlights the detection of LASV in non-*M. natalensis* species. *Mastomys erythroleucus* a morphologically indistinguishable, closely related species to *M. natalensis* has been found to co-occur with *M. natalensis* in several regions of Guinea, Sierra Leone and Nigeria [@brouat_phylogeography_2009]. The proportion of *M. erythroleucus* individuals found to be infected with LASV was not too dissimilar from *M. natalensis* and may indicate that this species can also be involved in viral transmission in locations where these species co-exist. Three other native rodent species *Mus minutoides*, *Mus baoulei* and *Hylomyscus pamfi* have been found to be acutely infected with LASV although the number of individuals of these tested is small. *Mus minutoides* and *Mus baoulei* are African pygmy mice a complex of 17-19 morphologically similar rodents that may contain a number of undescribed subspecies [@britton-davidian_systematics_2012]. They occupy a wide range of landuse types and are not considered synanthropic with a habitat preference for forests and shrubland, although they are often detected in cultivated landscapes [@long_multi-scale_2013]. Finally, the invasive rodent species, *Rattus rattus* has been found to be acutely infected with LASV, this synanthropic rodent species has been found to co-occur with *M. natalensis* in locations within which it has invaded and may represent a relatively recent host of LASV within the endemic region [@olayemi_small_2018; @bangura_lassa_2021].

More rodent species have been found to have antibodies to LASV within West Africa. Whether these species are competent hosts of the virus and can lead to onward rodent-to-rodent or rodent-to-human transmission is not known. Additionally, some of these detections may be due to the presence of cross-reactive antibodies to other Arenaviridae, although, a validated ELISA assay used for much of these surveys shows a sensitivity of 97.1% and specificity of 100% to LASV [@soubrier_detection_2022]. The proportion of all samples that tested positive for LASV antibodies across all species is typically lower, consistent with the greater number of samples assayed. Additional species found to have antibodies to LASV include the native rodents *Praomys daltoni*, *Praomys rostratus*, *Lemniscomys striatus*, *Lophuromys sikapusi* and *Gerbilliscus kempii*. Of these species only *P. daltoni* is considered synanthropic, typically found in villages and nearby agricultural areas in West Africa [@nicolas_comparative_2008; @mikula_commensalism_2020]. *P. rostratus* and *L. sikapusi* are more commonly found in forested or fragmented forest, shrubland and agricultural habitats [@felix_houphouet-boigny_university_cote_divoire_terrestrial_2018; @iyawe_distribution_1988]. *L. striatus* and *G. kempii* are considered savannah rodents, rarely detected within villages but often detected in forested habitats, shrubland and agriculture [@hoffmann_spatial_2001; @lourie_isolation_1975; @davis_affinities_1949]. Finally, non-rodent species found to have antibodies to LASV include individuals of the species rich insectivorous shrew order (*Crocidura spp.*), morphological identification to species level is typically not performed, hence, the grouping at order level.


Two invasive rodent species, *Mus musculus* and *R. rattus* are increasingly common in West Africa. These species have been introduced through human activity, typically in coastal regions, beginning in the 15th century with subsequent expansion into the interior of countries along human transport networks [@dalecky_range_2015]. Populations of these species have been found to establish in areas of human habitation demonstrating their synanthropic properties [@puckett_commensal_2020; @hima_native_2019]. These species appear to have different effects on local species richness following establishment, *M. musculus* but not *R. rattus* leads to reduced rodent species richness in locations in which it is detected [@dalecky_range_2015]. This may have important implications for the prevalence of LASV in the endemic region, if displacement of the primary reservoir by these invasive species that are potentially less competent hosts of viral transmission the risk of Lassa fever outbreaks will decrease.


Sampling of rodent species and locations of confirmed human cases have been used to produce risk maps of Lassa fever outbreaks. Several studies have produced risk maps, either based on human cases or *M. natalensis* occurrence that consistently identify the Mano River region and Nigeria as hotspots of risk [@basinski_bridging_2021; @gibb_understanding_2017; @mylne_mapping_2015; @fichet-calvet_risk_2009]. The studies are conducted at the regional scale and are not able to incorporate the heterogeneity of rodent species occurrence or abundance that has been observed in rodent sampling studies. The potential contribution of wider rodent communities to viral transmission or maintenance is not incorporated in these models that focus on *M. natalensis* as the sole reservoir of LASV in the endemic region. Additionally, sampling of rodents and LASV in West Africa is biased taxonomically and geographically with increased sampling effort in locations with historical outbreaks of Lassa fever [@beck_spatial_2014; @klitting_predicting_2022]. The association of sampling with locations of known outbreaks may be artificially biasing risk to these locations, with suggestion that risk is more evenly distributed across West Africa [@peterson_mapping_2014]. A better understanding of sampling biases in both small mammal communities and pathogen sampling will assist in identifying regions in which additional sampling is required. Further, inferring change in risk predicted to ensue under global change is limited by currently biased data [@boria_spatial_2014; @wille_how_2021].


In summary Lassa fever is a zoonotic infectious disease with regular outbreaks in several countries of West Africa, but is likely to be endemic across the region. This pathogen has been found to infect multiple rodent species, including both synanthropic and non-commensal rodent species, suggesting a role of the wider rodent community in maintenance of the pathogen in endemic regions. To better understand the current and future risk of Lassa fever spillover biases in available data need to be characterised and systematic sampling of the entire rodent community is required. 


## Thesis aims and structure


This thesis will aim to address some of the critical gaps in understanding of LASV transmission among rodent communities in endemic settings and the effect of anthropogenic change on the structure of rodent communities and how this may modulate Lassa fever spillover risk. The first part of this thesis (Chapter 2) attempts to understand a key problem in the sampling of both rodents and their pathogens across West Africa. Can sampling sampling bias of rodents and their pathogens be quantified and mitigated against? To achieve this I synthesise rodent trapping studies in the region and assess spatial biases, identifying regions that have been relatively under-sampled and therefore locations in which inference may be limited based on currently available datasets. I compare this constructed dataset to a commonly used resource, the Global Biodiversity Information Facility (GBIF), to explore the benefit of incorporating primary rodent trapping data with this consolidated dataset. The results presented in this chapter and the planned incorporation of this data into GBIF will aid researchers attempting to model risks of rodent borne zoonoses in these settings and the effect of future anthropogenic change on rodent distributions across the region.


The second part of this thesis (Chapter 3 and 4) present the results of a systematic study of rodent ecology and LASV prevalence in a Lassa fever endemic region of Eastern Sierra Leone. Chapter 3 describes the composition of rodent communities in Eastern Sierra Leone and aims to answer the question does rodent species richness and diversity vary along an anthropogenic landuse gradient? This study explores the biotic interactions between rodent species to infer the risk of LASV transmission among rodent communities along a landuse gradient using species occupancy models incorporating incomplete detection. Chapter 4 expands on this work to explicitly model potential contact networks among individual rodents in different landuse types to investigate the interactions within these rodent communities. I ask whether the primary reservoir is more likely to interact with members of the same species and what this may mean for viral transmission. I use Exponential Random Graph Models fitted to produced networks of rodent contacts based on rodent trapping data to assess the probability of inter- and intra-specific contact rates to understand viral transmission within rodent communities. Within this chapter I describe the prevalence of antibodies to LASV to gauge the risk of Lassa fever spillover in these settings. 


The thesis concludes with a discussion of how insights from this body of work enhance our understanding of the risk of rodent borne zoonoses in West Africa and more specifically Lassa fever emergence in Sierra Leone. I identify future directions of study required to better quantify this dynamic risk and how these can guide timely public health interventions to reduce disease associated morbidity and mortality. 


\newpage